<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Publications | Andersson Lab </title> <meta name="author" content="Andersson Lab"> <meta name="description" content="Selected publications in reverse chronological order."> <meta name="keywords" content="Robin Andersson, Andersson Lab, enhancers, gene regulation, transcriptional regulation, University of Copenhagen"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://anderssonlab.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Andersson Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">Research </a> </li> <li class="nav-item "> <a class="nav-link" href="/team/">Team </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/data/">Data </a> </li> <li class="nav-item "> <a class="nav-link" href="/software/">Software </a> </li> <li class="nav-item "> <a class="nav-link" href="/contact/">Contact </a> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description">Selected publications in reverse chronological order.</p> </header> <article> <div class="social" style="text-align: left"> <div class="contact-icons"> <a href="https://scholar.google.com/citations?user=D8RKCXsAAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://www.ncbi.nlm.nih.gov/myncbi/robin.andersson.1/bibliography/public/" title="PubMed" rel="external nofollow noopener" target="_blank"><i class="ai ai-pubmed"></i></a> </div> </div> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Sheth2024-ln" class="col-sm-10"> <div class="title">Mapping enhancer-gene regulatory interactions from single-cell data</div> <div class="author"> Maya U Sheth, Wei-Lin Qiu, X Rosa Ma, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Andreas R Gschwind, Evelyn Jagoda, Anthony S Tan, Hjörleifur Einarsson, Bram L Gorissen, Danilo Dubocanin, Christopher S McGinnis, Dulguun Amgalan, Ansuman T Satpathy, Thouis R Jones, Lars M Steinmetz, Anshul Kundaje, Berk Ustun, Jesse M Engreitz, Robin Andersson' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>bioRxivorg</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.11.23.624931" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Mapping enhancers and their target genes in specific cell types is crucial for understanding gene regulation and human disease genetics. However, accurately predicting enhancer-gene regulatory interactions from single-cell datasets has been challenging. Here, we introduce a new family of classification models, scE2G, to predict enhancer-gene regulation. These models use features from single-cell ATAC-seq or multiomic RNA and ATAC-seq data and are trained on a CRISPR perturbation dataset including &gt;10,000 evaluated element-gene pairs. We benchmark scE2G models against CRISPR perturbations, fine-mapped eQTLs, and GWAS variant-gene associations and demonstrate state-of-the-art performance at prediction tasks across multiple cell types and categories of perturbations. We apply scE2G to build maps of enhancer-gene regulatory interactions in heterogeneous tissues and interpret noncoding variants associated with complex traits, nominating regulatory interactions linking INPP4B and IL15 to lymphocyte counts. The scE2G models will enable accurate mapping of enhancer-gene regulatory interactions across thousands of diverse human cell types.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Ma2024-dr" class="col-sm-10"> <div class="title">Molecular convergence of risk variants for congenital heart defects leveraging a regulatory map of the human fetal heart</div> <div class="author"> X Rosa Ma, Stephanie D Conley, Michael Kosicki, and <span class="more-authors" title="click to view 34 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '34 more authors' ? 'Danila Bredikhin, Ran Cui, Steven Tran, Maya U Sheth, Wei-Lin Qiu, Sijie Chen, Soumya Kundu, Helen Y Kang, Dulguun Amgalan, Chad J Munger, Lauren Duan, Katherine Dang, Oriane Matthys Rubio, Shinwan Kany, Siavash Zamirpour, John DePaolo, Arun Padmanabhan, Birth Defects Research Laboratory, Jeffrey Olgin, Scott Damrauer, Robin Andersson, Mingxia Gu, James R Priest, Thomas Quertermous, Xiaojie Qiu, Marlene Rabinovitch, Axel Visel, Len Pennacchio, Anshul Kundaje, Ian A Glass, Casey A Gifford, James P Pirruccello, William R Goodyer, Jesse M Engreitz' : '34 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">34 more authors</span> </div> <div class="periodical"> <em>medRxiv</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.11.20.24317557" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Congenital heart defects (CHD) arise in part due to inherited genetic variants that alter genes and noncoding regulatory elements in the human genome. These variants are thought to act during fetal development to influence the formation of different heart structures. However, identifying the genes, pathways, and cell types that mediate these effects has been challenging due to the immense diversity of cell types involved in heart development as well as the superimposed complexities of interpreting noncoding sequences. As such, understanding the molecular functions of both noncoding and coding variants remains paramount to our fundamental understanding of cardiac development and CHD. Here, we created a gene regulation map of the healthy human fetal heart across developmental time, and applied it to interpret the functions of variants associated with CHD and quantitative cardiac traits. We collected single-cell multiomic data from 734,000 single cells sampled from 41 fetal hearts spanning post-conception weeks 6 to 22, enabling the construction of gene regulation maps in 90 cardiac cell types and states, including rare populations of cardiac conduction cells. Through an unbiased analysis of all 90 cell types, we find that both rare coding variants associated with CHD and common noncoding variants associated with valve traits converge to affect valvular interstitial cells (VICs). VICs are enriched for high expression of known CHD genes previously identified through mapping of rare coding variants. Eight CHD genes, as well as other genes in similar molecular pathways, are linked to common noncoding variants associated with other valve diseases or traits via enhancers in VICs. In addition, certain common noncoding variants impact enhancers with activities highly specific to particular subanatomic structures in the heart, illuminating how such variants can impact specific aspects of heart structure and function. Together, these results implicate new enhancers, genes, and cell types in the genetic etiology of CHD, identify molecular convergence of common noncoding and rare coding variants on VICs, and suggest a more expansive view of the cell types instrumental in genetic risk for CHD, beyond the working cardiomyocyte. This regulatory map of the human fetal heart will provide a foundational resource for understanding cardiac development, interpreting genetic variants associated with heart disease, and discovering targets for cell-type specific therapies.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Amrute2024-se" class="col-sm-10"> <div class="title">Single cell variant to enhancer to gene map for coronary artery disease</div> <div class="author"> Junedh M Amrute, Paul C Lee, Ittai Eres, and <span class="more-authors" title="click to view 31 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '31 more authors' ? 'Chang Jie Mick Lee, Andrea Bredemeyer, Maya U Sheth, Tracy Yamawaki, Rijan Gurung, Chukwuemeka Anene-Nzelu, Wei-Lin Qiu, Soumya Kundu, Daniel Y Li, Markus Ramste, Daniel Lu, Anthony Tan, Chul-Joo Kang, Ryan E Wagoner, Arturo Alisio, Paul Cheng, Quanyi Zhao, Clint L Miller, Ira M Hall, Rajat M Gupta, Yi-Hsiang Hsu, Saptarsi M Haldar, Kory J Lavine, Simon Jackson, Robin Andersson, Jesse M Engreitz, Roger S-Y Foo, Chi-Ming Li, Brandon Ason, Thomas Quertermous, Nathan O Stitziel' : '31 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">31 more authors</span> </div> <div class="periodical"> <em>medRxiv</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.11.13.24317257" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>AbstractAlthough genome wide association studies (GWAS) in large populations have identified hundreds of variants associated with common diseases such as coronary artery disease (CAD), most disease-associated variants lie within non-coding regions of the genome, rendering it difficult to determine the downstream causal gene and cell type. Here, we performed paired single nucleus gene expression and chromatin accessibility profiling from 44 human coronary arteries. To link disease variants to molecular traits, we developed a meta-map of 88 samples and discovered 11,182 single-cell chromatin accessibility quantitative trait loci (caQTLs). Heritability enrichment analysis and disease variant mapping demonstrated that smooth muscle cells (SMCs) harbor the greatest genetic risk for CAD. To capture the continuum of SMC cell states in disease, we used dynamic single cell caQTL modeling for the first time in tissue to uncover QTLs whose effects are modified by cell state and expand our insight into genetic regulation of heterogenous cell populations. Notably, we identified a variant in theCOL4A1/COL4A2CAD GWAS locus which becomes a caQTL as SMCs de-differentiate by changing a transcription factor binding site for EGR1/2. To unbiasedly prioritize functional candidate genes, we built a genome-wide single cell variant to enhancer to gene (scV2E2G) map for human CAD to link disease variants to causal genes in cell types. Using this approach, we found several hundred genes predicted to be linked to disease variants in different cell types. Next, we performed genome-wide Hi-C in 16 human coronary arteries to build tissue specific maps of chromatin conformation and link disease variants to integrated chromatin hubs and distal target genes. Using this approach, we show that rs4887091 within theADAMTS7CAD GWAS locus modulates function of a super chromatin interactome through a change in a CTCF binding site. Finally, we used CRISPR interference to validate a distal gene,AMOTL2, liked to a CAD GWAS locus. Collectively we provide a disease-agnostic framework to translate human genetic findings to identify pathologic cell states and genes driving disease, producing a comprehensive scV2E2G map with genetic and tissue level convergence for future mechanistic and therapeutic studies.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Jakobsen2024-vc" class="col-sm-10"> <div class="title">Publisher Correction: MYC activity at enhancers drives prognostic transcriptional programs through an epigenetic switch</div> <div class="author"> Simon T Jakobsen, Rikke A M Jensen, Maria S Madsen, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Tina Ravnsborg, Christian S Vaagenso, Majken S Siersbæk, Hjorleifur Einarsson, Robin Andersson, Ole N Jensen, Rasmus Siersbæk' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Nat. Genet.</em>, Aug 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/s41588-024-01856-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Stigliani2024-ju" class="col-sm-10"> <div class="title">Adaptation to an acid microenvironment promotes pancreatic cancer organoid growth and drug resistance</div> <div class="author"> Arnaud Stigliani, Renata Ialchina, Jiayi Yao, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Dominika Czaplinska, Yifan Dai, Henriette Berg Andersen, Sarah Rennie, Robin Andersson, Stine Falsig Pedersen, Albin Sandelin' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Cell Rep.</em>, Jul 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2024.114409" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Harsh environments in poorly perfused tumor regions may select for traits driving cancer aggressiveness. Here, we investigated whether tumor acidosis interacts with driver mutations to exacerbate cancer hallmarks. We adapted mouse organoids from normal pancreatic duct (mN10) and early pancreatic cancer (mP4, KRAS-G12D mutation, ± p53 knockout) from extracellular pH 7.4 to 6.7, representing acidic niches. Viability was increased by acid adaptation, a pattern most apparent in wild-type (WT) p53 organoids, and exacerbated upon return to pH 7.4. This led to increased survival of acid-adapted organoids treated with gemcitabine and/or erlotinib, and, in WT p53 organoids, acid-induced attenuation of drug effects. New genetic variants became dominant during adaptation, yet they were unlikely to be its main drivers. Transcriptional changes induced by acid and drug adaptation differed overall, but acid adaptation increased the expression of gemcitabine resistance genes. Thus, adaptation to acidosis increases cancer cell viability after chemotherapy.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Jakobsen2024-lq" class="col-sm-10"> <div class="title">MYC activity at enhancers drives prognostic transcriptional programs through an epigenetic switch</div> <div class="author"> Simon T Jakobsen, Rikke A M Jensen, Maria S Madsen, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Tina Ravnsborg, Christian S Vaagenso, Majken S Siersbæk, Hjorleifur Einarsson, Robin Andersson, Ole N Jensen, Rasmus Siersbæk' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Nat. Genet.</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41588-024-01676-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The transcription factor MYC is overexpressed in most cancers, where it drives multiple hallmarks of cancer progression. MYC is known to promote oncogenic transcription by binding to active promoters. In addition, MYC has also been shown to invade distal enhancers when expressed at oncogenic levels, but this enhancer binding has been proposed to have low gene-regulatory potential. Here, we demonstrate that MYC directly regulates enhancer activity to promote cancer type-specific gene programs predictive of poor patient prognosis. MYC induces transcription of enhancer RNA through recruitment of RNA polymerase II (RNAPII), rather than regulating RNAPII pause-release, as is the case at promoters. This process is mediated by MYC-induced H3K9 demethylation and acetylation by GCN5, leading to enhancer-specific BRD4 recruitment through its bromodomains, which facilitates RNAPII recruitment. We propose that MYC drives prognostic cancer type-specific gene programs through induction of an enhancer-specific epigenetic switch, which can be targeted by BET and GCN5 inhibitors.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Wenger2023-bp" class="col-sm-10"> <div class="title">Symmetric inheritance of parental histones governs epigenome maintenance and embryonic stem cell identity</div> <div class="author"> Alice Wenger, Alva Biran, Nicolas Alcaraz, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Alba Redó-Riveiro, Annika Charlotte Sell, Robert Krautz, Valentin Flury, Nazaret Reverón-Gómez, Victor Solis-Mezarino, Moritz Völker-Albert, Axel Imhof, Robin Andersson, Joshua M Brickman, Anja Groth' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>Nat. Genet.</em>, Sep 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41588-023-01476-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Modified parental histones are segregated symmetrically to daughter DNA strands during replication and can be inherited through mitosis. How this may sustain the epigenome and cell identity remains unknown. Here we show that transmission of histone-based information during DNA replication maintains epigenome fidelity and embryonic stem cell plasticity. Asymmetric segregation of parental histones H3-H4 in MCM2-2A mutants compromised mitotic inheritance of histone modifications and globally altered the epigenome. This included widespread spurious deposition of repressive modifications, suggesting elevated epigenetic noise. Moreover, H3K9me3 loss at repeats caused derepression and H3K27me3 redistribution across bivalent promoters correlated with misexpression of developmental genes. MCM2-2A mutation challenged dynamic transitions in cellular states across the cell cycle, enhancing naïve pluripotency and reducing lineage priming in G1. Furthermore, developmental competence was diminished, correlating with impaired exit from pluripotency. Collectively, this argues that epigenetic inheritance of histone modifications maintains a correctly balanced and dynamic chromatin landscape able to support mammalian cell differentiation.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="van-Duin2023-jc" class="col-sm-10"> <div class="title">Transcription factor expression is the main determinant of variability in gene co-activity</div> <div class="author"> Lucas Duin, Robert Krautz, Sarah Rennie, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Robin Andersson' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Mol. Syst. Biol.</em>, Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.15252/msb.202211392" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Many genes are co-expressed and form genomic domains of coordinated gene activity. However, the regulatory determinants of domain co-activity remain unclear. Here, we leverage human individual variation in gene expression to characterize the co-regulatory processes underlying domain co-activity and systematically quantify their effect sizes. We employ transcriptional decomposition to extract from RNA expression data an expression component related to co-activity revealed by genomic positioning. This strategy reveals close to 1,500 co-activity domains, covering most expressed genes, of which the large majority are invariable across individuals. Focusing specifically on domains with high variability in co-activity reveals that contained genes have a higher sharing of eQTLs, a higher variability in enhancer interactions, and an enrichment of binding by variably expressed transcription factors, compared to genes within non-variable domains. Through careful quantification of the relative contributions of regulatory processes underlying co-activity, we find transcription factor expression levels to be the main determinant of gene co-activity. Our results indicate that distal trans effects contribute more than local genetic variation to individual variation in co-activity domains.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Mei2023-hs" class="col-sm-10"> <div class="title">Pooled analysis of frontal lobe transcriptomic data identifies key mitophagy gene changes in Alzheimer’s disease brain</div> <div class="author"> Taoyu Mei, Yuan Li, Anna Orduña Dolado, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Zhiquan Li, Robin Andersson, Laura Berliocchi, Lene Juel Rasmussen' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Front. Aging Neurosci.</em>, Jun 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fnagi.2023.1101216" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Background: The growing prevalence of Alzheimer’s disease (AD) is becoming a global health challenge without effective treatments. Defective mitochondrial function and mitophagy have recently been suggested as etiological factors in AD, in association with abnormalities in components of the autophagic machinery like lysosomes and phagosomes. Several large transcriptomic studies have been performed on different brain regions from AD and healthy patients, and their data represent a vast source of important information that can be utilized to understand this condition. However, large integration analyses of these publicly available data, such as AD RNA-Seq data, are still missing. In addition, large-scale focused analysis on mitophagy, which seems to be relevant for the aetiology of the disease, has not yet been performed. Methods: In this study, publicly available raw RNA-Seq data generated from healthy control and sporadic AD post-mortem human samples of the brain frontal lobe were collected and integrated. Sex-specific differential expression analysis was performed on the combined data set after batch effect correction. From the resulting set of differentially expressed genes, candidate mitophagy-related genes were identified based on their known functional roles in mitophagy, the lysosome, or the phagosome, followed by Protein-Protein Interaction (PPI) and microRNA-mRNA network analysis. The expression changes of candidate genes were further validated in human skin fibroblast and induced pluripotent stem cells (iPSCs)-derived cortical neurons from AD patients and matching healthy controls. Results: From a large dataset (AD: 589; control: 246) based on three different datasets (i.e., ROSMAP, MSBB, &amp; GSE110731), we identified 299 candidate mitophagy-related differentially expressed genes (DEG) in sporadic AD patients (male: 195, female: 188). Among these, the AAA ATPase VCP, the GTPase ARF1, the autophagic vesicle forming protein GABARAPL1 and the cytoskeleton protein actin beta ACTB were selected based on network degrees and existing literature. Changes in their expression were further validated in AD-relevant human in vitro models, which confirmed their down-regulation in AD conditions. Conclusion: Through the joint analysis of multiple publicly available data sets, we identify four differentially expressed key mitophagy-related genes potentially relevant for the pathogenesis of sporadic AD. Changes in expression of these four genes were validated using two AD-relevant human in vitro models, primary human fibroblasts and iPSC-derived neurons. Our results provide foundation for further investigation of these genes as potential biomarkers or disease-modifying pharmacological targets.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Salvatore2023-bs" class="col-sm-10"> <div class="title">Transfer learning identifies sequence determinants of cell-type specific regulatory element accessibility</div> <div class="author"> Marco Salvatore, Marc Horlacher, Annalisa Marsico, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Ole Winther, Robin Andersson' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>NAR Genom. Bioinform.</em>, Jun 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/nargab/lqad026" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Dysfunction of regulatory elements through genetic variants is a central mechanism in the pathogenesis of disease. To better understand disease etiology, there is consequently a need to understand how DNA encodes regulatory activity. Deep learning methods show great promise for modeling of biomolecular data from DNA sequence but are limited to large input data for training. Here, we develop ChromTransfer, a transfer learning method that uses a pre-trained, cell-type agnostic model of open chromatin regions as a basis for fine-tuning on regulatory sequences. We demonstrate superior performances with ChromTransfer for learning cell-type specific chromatin accessibility from sequence compared to models not informed by a pre-trained model. Importantly, ChromTransfer enables fine-tuning on small input data with minimal decrease in accuracy. We show that ChromTransfer uses sequence features matching binding site sequences of key transcription factors for prediction. Together, these results demonstrate ChromTransfer as a promising tool for learning the regulatory code.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Zaugg2022-qq" class="col-sm-10"> <div class="title">Current challenges in understanding the role of enhancers in disease</div> <div class="author"> Judith Barbara Zaugg, Pelin Sahlén, Robin Andersson, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Meritxell Alberich-Jorda, Wouter Laat, Bart Deplancke, Jorge Ferrer, Susanne Mandrup, Gioacchino Natoli, Dariusz Plewczynski, Alvaro Rada-Iglesias, Salvatore Spicuglia' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Nat. Struct. Mol. Biol.</em>, Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41594-022-00896-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Enhancers play a central role in the spatiotemporal control of gene expression and tend to work in a cell-type-specific manner. In addition, they are suggested to be major contributors to phenotypic variation, evolution and disease. There is growing evidence that enhancer dysfunction due to genetic, structural or epigenetic mechanisms contributes to a broad range of human diseases referred to as enhanceropathies. Such mechanisms often underlie the susceptibility to common diseases, but can also play a direct causal role in cancer or Mendelian diseases. Despite the recent gain of insights into enhancer biology and function, we still have a limited ability to predict how enhancer dysfunction impacts gene expression. Here we discuss the major challenges that need to be overcome when studying the role of enhancers in disease etiology and highlight opportunities and directions for future studies, aiming to disentangle the molecular basis of enhanceropathies.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Einarsson2022-xb" class="col-sm-10"> <div class="title">Promoter sequence and architecture determine expression variability and confer robustness to genetic variants</div> <div class="author"> Hjorleifur Einarsson, Marco Salvatore, Christian Vaagenso, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Nicolas Alcaraz, Jette Bornholdt Lange, Sarah Rennie, Robin Andersson' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Nov 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.80943" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Genetic and environmental exposures cause variability in gene expression. Although most genes are affected in a population, their effect sizes vary greatly, indicating the existence of regulatory mechanisms that could amplify or attenuate expression variability. Here, we investigate the relationship between the sequence and transcription start site architectures of promoters and their expression variability across human individuals. We find that expression variability can be largely explained by a promoter’s DNA sequence and its binding sites for specific transcription factors. We show that promoter expression variability reflects the biological process of a gene, demonstrating a selective trade-off between stability for metabolic genes and plasticity for responsive genes and those involved in signaling. Promoters with a rigid transcription start site architecture are more prone to have variable expression and to be associated with genetic variants with large effect sizes, while a flexible usage of transcription start sites within a promoter attenuates expression variability and limits genotypic effects. Our work provides insights into the variable nature of responsive genes and reveals a novel mechanism for supplying transcriptional and mutational robustness to essential genes through multiple transcription start site regions within a promoter.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Bakoulis2022-rg" class="col-sm-10"> <div class="title">Endogenous retroviruses co-opted as divergently transcribed regulatory elements shape the regulatory landscape of embryonic stem cells</div> <div class="author"> Stylianos Bakoulis, Robert Krautz, Nicolas Alcaraz, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Marco Salvatore, Robin Andersson' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Feb 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/nar/gkac088" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Transposable elements are an abundant source of transcription factor binding sites, and favorable genomic integration may lead to their recruitment by the host genome for gene regulatory functions. However, it is unclear how frequent co-option of transposable elements as regulatory elements is, to which regulatory programs they contribute and how they compare to regulatory elements devoid of transposable elements. Here, we report a transcription initiation-centric, in-depth characterization of the transposon-derived regulatory landscape of mouse embryonic stem cells. We demonstrate that a substantial number of transposable element insertions, in particular endogenous retroviral elements, are associated with open chromatin regions that are divergently transcribed into unstable RNAs in a cell-type specific manner, and that these elements contribute to a sizable proportion of active enhancers and gene promoters. We further show that transposon subfamilies contribute differently and distinctly to the pluripotency regulatory program through their repertoires of transcription factor binding site sequences, shedding light on the formation of regulatory programs and the origins of regulatory elements.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Rennie2021-kv" class="col-sm-10"> <div class="title">hyperTRIBER: a flexible R package for the analysis of differential RNA editing</div> <div class="author"> Sarah Rennie, Daniel Heidar Magnusson, and Robin Andersson </div> <div class="periodical"> Oct 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2021.10.20.465108v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>RNA editing by ADAR (adenosine deaminase acting on RNA) is gaining an increased interest in the field of post-transcriptional regulation. Fused to an RNA-binding protein (RBP) of interest, the catalytic activity of ADAR results in A-to-I RNA edits, whose identification will determine RBP-bound RNA transcripts. However, the computational tools available for their identification and differential RNA editing statistical analysis are limited or too specialised for general-purpose usage. Here we present hyperTRIBER, a flexible suite of tools, wrapped into a convenient R package, for the detection of differential RNA editing. hyperTRIBER is applicable to complex scenarios and experimental designs, and provides a robust statistical framework allowing for the control for coverage of reads at a given base, the total expression level and other co-variates. We demonstrate the capabilities of our approach on HyperTRIBE RNA-seq data for the detection of bound RNAs by the N6-methyladenosine (m6A) reader protein ECT2 in Arabidopsis roots. We show that hyperTRIBER finds edits with a high statistical power, even where editing proportions and RNA transcript expression levels are low, together demonstrating its usability and versatility for analysing differential RNA editing.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Arribas-Hernandez2021-bt" class="col-sm-10"> <div class="title">Principles of mRNA targeting via the Arabidopsis m6A-binding protein ECT2</div> <div class="author"> Laura Arribas-Hernández, Sarah Rennie, Tino Köster, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Carlotta Porcelli, Martin Lewinski, Prof Dorothee Staiger, Robin Andersson, Peter Brodersen' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.72375" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Specific recognition of N6-methyladenosine (m6A) in mRNA by RNA-binding proteins containing a YT521-B homology (YTH) domain is important in eukaryotic gene regulation. The Arabidopsis YTH-domain protein ECT2 is thought to bind to mRNA at URU(m6A)Y sites, yet RR(m6A)CH is the canonical m6A consensus site in all eukaryotes and ECT2 functions require m6A binding activity. Here, we apply iCLIP (individual-nucleotide resolution cross-linking and immunoprecipitation) and HyperTRIBE (targets of RNA-binding proteins identified by editing) to define high-quality target sets of ECT2, and analyze the patterns of enriched sequence motifs around ECT2 crosslink sites. Our analyses show that ECT2 does in fact bind to RR(m6A)CH. Pyrimidine-rich motifs are enriched around, but not at m6A-sites, reflecting a preference for N6-adenosine methylation of RRACH/GGAU islands in pyrimidine-rich regions. Such motifs, particularly oligo-U and UNUNU upstream of m6A sites, are also implicated in ECT2 binding via its intrinsically disordered region (IDR). Finally, URUAY-type motifs are enriched at ECT2 crosslink sites, but their distinct properties suggest function as sites of competition between binding of ECT2 and as yet unidentified RNA-binding proteins. Our study provides coherence between genetic and molecular studies of m6A-YTH function in plants, and reveals new insight into the mode of RNA recognition by YTH-domain-containing proteins.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Arribas-Hernandez2021-en" class="col-sm-10"> <div class="title">The YTHDF proteins ECT2 and ECT3 bind largely overlapping target sets and influence target mRNA abundance, not alternative polyadenylation</div> <div class="author"> Laura Arribas-Hernández, Sarah Rennie, Michael Schon, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Carlotta Porcelli, Balaji Enugutti, Robin Andersson, Michael D Nodine, Peter Brodersen' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.72377" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Gene regulation via N6-methyladenosine (m6A) in mRNA involves RNA-binding proteins that recognize m6A via a YT521-B homology (YTH) domain. The plant YTH domain proteins ECT2 and ECT3 act genetically redundantly in stimulating cell proliferation during organogenesis, but several fundamental questions regarding their mode of action remain unclear. Here, we use HyperTRIBE (targets of RNA-binding proteins identified by editing) to show that most ECT2 and ECT3 targets overlap, with only few examples of preferential targeting by either of the two proteins. HyperTRIBE in different mutant backgrounds also provides direct views of redundant and specific target interactions of the two proteins. We also show that contrary to conclusions of previous reports, ECT2 does not accumulate in the nucleus. Accordingly, inactivation of ECT2, ECT3 and their surrogate ECT4 does not change patterns of polyadenylation site choice in ECT2/3 target mRNAs, but does lead to lower steady state accumulation of target mRNAs. In addition, mRNA and microRNA expression profiles show indications of stress response activation in ect2/ect3/ect4 mutants, likely via indirect effects. Thus, previous suggestions of control of alternative polyadenylation by ECT2 are not supported by evidence, and ECT2 and ECT3 act largely redundantly to regulate target mRNA, including its abundance, in the cytoplasm.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Petryk2021-yq" class="col-sm-10"> <div class="title">Genome-wide and sister chromatid-resolved profiling of protein occupancy in replicated chromatin with ChOR-seq and SCAR-seq | Nature Protocols</div> <div class="author"> Nataliya Petryk, Nazaret Reverón-Gómez, Cristina González-Aguilera, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Maria Dalby, Robin Andersson, Anja Groth' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41596-021-00585-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Elucidating the mechanisms underlying chromatin maintenance upon genome replication is critical for the understanding of how gene expression programs and cell identity are preserved across cell divisions. Here, we describe two recently developed techniques, chromatin occupancy after replication (ChOR)-seq and sister chromatids after replication (SCAR)-seq, that profile chromatin occupancy on newly replicated DNA in mammalian cells in 5 d of bench work. Both techniques share a common strategy that includes pulse labeling of newly synthesized DNA and chromatin immunoprecipitation (ChIP), followed by purification and high-throughput sequencing. Whereas ChOR-seq quantitatively profiles the post-replicative abundance of histone modifications and chromatin-associated proteins, SCAR-seq distinguishes chromatin occupancy between nascent sister chromatids. Together, these two complementary techniques have unraveled key mechanisms controlling the inheritance of modified histones during replication and revealed locus-specific dynamics of histone modifications across the cell cycle. Here, we provide the experimental protocols and bioinformatic pipelines for these methods.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Alam2020-px" class="col-sm-10"> <div class="title">Comparative transcriptomics of primary cells in vertebrates</div> <div class="author"> Tanvir Alam, Saumya Agrawal, Jessica Severin, and <span class="more-authors" title="click to view 22 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '22 more authors' ? 'Robert S Young, Robin Andersson, Erik Arner, Akira Hasegawa, Marina Lizio, Jordan A Ramilowski, Imad Abugessaisa, Yuri Ishizu, Shohei Noma, Hiroshi Tarui, Martin S Taylor, Timo Lassmann, Masayoshi Itoh, Takeya Kasukawa, Hideya Kawaji, Luigi Marchionni, Guojun Sheng, Alistair R R Forrest, Levon M Khachigian, Yoshihide Hayashizaki, Piero Carninci, Michiel J L de Hoon' : '22 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">22 more authors</span> </div> <div class="periodical"> <em>Genome Res.</em>, Jul 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/gr.255679.119" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Gene expression profiles in homologous tissues have been observed to be different between species, which may be due to differences between species in the gene expression program in each cell type, but may also reflect differences in cell type composition of each tissue in different species. Here, we compare expression profiles in matching primary cells in human, mouse, rat, dog, and chicken using Cap Analysis Gene Expression (CAGE) and short RNA (sRNA) sequencing data from FANTOM5. While we find that expression profiles of orthologous genes in different species are highly correlated across cell types, in each cell type many genes were differentially expressed between species. Expression of genes with products involved in transcription, RNA processing, and transcriptional regulation was more likely to be conserved, while expression of genes encoding proteins involved in intercellular communication was more likely to have diverged during evolution. Conservation of expression correlated positively with the evolutionary age of genes, suggesting that divergence in expression levels of genes critical for cell function was restricted during evolution. Motif activity analysis showed that both promoters and enhancers are activated by the same transcription factors in different species. An analysis of expression levels of mature miRNAs and of primary miRNAs identified by CAGE revealed that evolutionary old miRNAs are more likely to have conserved expression patterns than young miRNAs. We conclude that key aspects of the regulatory network are conserved, while differential expression of genes involved in cell-to-cell communication may contribute greatly to phenotypic differences between species.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Andersson2020-xi" class="col-sm-10"> <div class="title">Determinants of enhancer and promoter activities of regulatory elements</div> <div class="author"> Robin Andersson, and Albin Sandelin </div> <div class="periodical"> <em>Nat. Rev. Genet.</em>, Feb 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41576-019-0173-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The proper activities of enhancers and gene promoters are essential for coordinated transcription within a cell. Although diverse methodologies have been developed to identify enhancers and promoters, most have tacitly assumed that these elements are distinct. However, studies have unexpectedly shown that regulatory elements may have both enhancer and promoter functions. Here we review these results, focusing on the factors that determine the promoter and/or enhancer activity of regulatory elements. We discuss emerging models that define regulatory elements by accessible DNA and their non-mutually-exclusive abilities to drive transcription initiation (promoter activity) and/or to enhance transcription at other such regions (enhancer activity).</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Gazova2020-zk" class="col-sm-10"> <div class="title">The Transcriptional Network That Controls Growth Arrest and Macrophage Differentiation in the Human Myeloid Leukemia Cell Line THP-1</div> <div class="author"> Iveta Gažová, Lucas Lefevre, Stephen J Bush, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Sara Clohisey, Erik Arner, Michiel Hoon, Jessica Severin, Lucas Duin, Robin Andersson, Andreas Lengeling, David A Hume, Kim M Summers' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Frontiers in Cell and Developmental Biology</em>, Feb 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fcell.2020.00498" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The response of the human acute myeloid leukemia cell line THP-1 to phorbol esters has been widely-studied to test candidate leukemia therapies and as a model of cell cycle arrest and monocyte-macrophage differentiation. Here we have employed Cap Analysis of Gene Expression (CAGE) to analyse a dense time course of transcriptional regulation in THP-1 cells treated with phorbol myristate acetate (PMA) over 96 hours. PMA treatment greatly reduced the numbers of cells entering S phase and also blocked cells exiting G2/M. The PMA-treated cells became adherent and expression of mature macrophage-specific genes increased progressively over the duration of the time course. Within 1-2 hours PMA induced known targets of tumour protein p53 (TP53), notably CDKN1A, followed by gradual down-regulation of cell-cycle associated genes. Also within the first 2 hours, PMA induced immediate early genes including transcription factor genes encoding proteins implicated in macrophage differentiation (EGR2, JUN, MAFB) and down-regulated genes for transcription factors involved in immature myeloid cell proliferation (MYB, IRF8, GFI1). The dense time course revealed that the response to PMA was not linear and progressive. Rather, network-based clustering of the time course data highlighted a sequential cascade of transient up- and down-regulated expression of genes encoding feedback regulators, as well as transcription factors associated with macrophage differentiation and their inferred target genes. CAGE also identified known and candidate novel enhancers expressed in THP-1 cells and many novel inducible genes that currently lack functional annotation and/or had no previously known function in macrophages. The time course is available on the ZENBU platform allowing comparison to FANTOM4 and FANTOM5 data.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Thodberg2019-sl" class="col-sm-10"> <div class="title">CAGEfightR: analysis of 5’-end data using R/Bioconductor</div> <div class="author"> Malte Thodberg, Axel Thieffry, Kristoffer Vitting-Seerup, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Robin Andersson, Albin Sandelin' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>BMC Bioinformatics</em>, Oct 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1186/s12859-019-3029-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>BACKGROUND: 5’-end sequencing assays, and Cap Analysis of Gene Expression (CAGE) in particular, have been instrumental in studying transcriptional regulation. 5’-end methods provide genome-wide maps of transcription start sites (TSSs) with base pair resolution. Because active enhancers often feature bidirectional TSSs, such data can also be used to predict enhancer candidates. The current availability of mature and comprehensive computational tools for the analysis of 5’-end data is limited, preventing efficient analysis of new and existing 5’-end data. RESULTS: We present CAGEfightR, a framework for analysis of CAGE and other 5’-end data implemented as an R/Bioconductor-package. CAGEfightR can import data from BigWig files and allows for fast and memory efficient prediction and analysis of TSSs and enhancers. Downstream analyses include quantification, normalization, annotation with transcript and gene models, TSS shape statistics, linking TSSs to enhancers via co-expression, identification of enhancer clusters, and genome-browser style visualization. While built to analyze CAGE data, we demonstrate the utility of CAGEfightR in analyzing nascent RNA 5’-data (PRO-Cap). CAGEfightR is implemented using standard Bioconductor classes, making it easy to learn, use and combine with other Bioconductor packages, for example popular differential expression tools such as limma, DESeq2 and edgeR. CONCLUSIONS: CAGEfightR provides a single, scalable and easy-to-use framework for comprehensive downstream analysis of 5’-end data. CAGEfightR is designed to be interoperable with other Bioconductor packages, thereby unlocking hundreds of mature transcriptomic analysis tools for 5’-end data. CAGEfightR is freely available via Bioconductor: bioconductor.org/packages/CAGEfightR .</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Singh2019-aq" class="col-sm-10"> <div class="title">PLZF targets developmental enhancers for activation during osteogenic differentiation of human mesenchymal stem cells</div> <div class="author"> Shuchi Agrawal Singh, Mads Lerdrup, Ana-Luisa R Gomes, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Harmen J G Werken, Jens Vilstrup Johansen, Robin Andersson, Albin Sandelin, Kristian Helin, Klaus Hansen' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Jan 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.40364" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A key transcription-factor for osteogenic differentiation, PLZF, acts as a transcriptional activator by binding to active developmental enhancers and facilitates mediator recruitment, but is not involved in enhancer looping.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Jefimov2018-eq" class="col-sm-10"> <div class="title">The GBAF chromatin remodeling complex binds H3K27ac and mediates enhancer transcription</div> <div class="author"> Kirill Jefimov, Nicolas Alcaraz, Susan Kloet, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Signe Varv, Siri Aastedatter Sakya, Christian Dalager Vaagenso, Michiel Vermeulen, Rein Aasland, Robin Andersson' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, Oct 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/445148" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>H3K27ac is associated with regulatory active enhancers, but its exact role in enhancer function remains elusive. Using mass spectrometry-based interaction proteomics, we identified the Super Elongation Complex (SEC) and GBAF, a non-canonical GLTSCR1L- and BRD9-containing SWI/SNF chromatin remodeling complex, to be major interactors of H3K27ac. We systematically characterized the composition of GBAF and the conserved GLTSCR1/1L 9GiBAF9-domain, which we found to be responsible for GBAF complex formation and GLTSCR1L nuclear localization. Inhibition of the bromodomain of BRD9 revealed interaction between GLTSCR1L and H3K27ac to be BRD9-dependent and led to GLTSCR1L dislocation from its preferred binding sites at H3K27ac-associated enhancers. GLTSCR1L disassociation from chromatin resulted in genome-wide downregulation of enhancer transcription while leaving most mRNA expression levels unchanged, except for reduced mRNA levels from loci topologically linked to affected enhancers. Our results indicate that GBAF is an enhancer-associated chromatin remodeler important for transcriptional and regulatory activity of enhancers.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Petryk2018-qg" class="col-sm-10"> <div class="title">MCM2 promotes symmetric inheritance of modified histones during DNA replication</div> <div class="author"> Nataliya Petryk, Maria Dalby, Alice Wenger, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Caroline B Stromme, Anne Strandsby, Robin Andersson, Anja Groth' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Science</em>, Sep 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.aau0294" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Parental histones with modifications are recycled to newly replicated DNA strands during genome replication, but do the two sister chromatids inherit modified histones equally? Yu et al. and Petryk et al. found in mouse and yeast, respectively, that modified histones are segregated to both DNA daughter strands in a largely symmetric manner (see the Perspective by Ahmad and Henikoff). However, the mechanisms ensuring this symmetric inheritance in yeast and mouse were different. Yeasts use subunits of DNA polymerase to prevent the lagging-strand bias of parental histones, whereas in mouse cells, the replicative helicase MCM2 counters the leading-strand bias. Science , this issue p. [1386][1], p. [1389][2]; see also p. [1311][3] During genome replication, parental histones are recycled to newly replicated DNA with their posttranslational modifications (PTMs). Whether sister chromatids inherit modified histones evenly remains unknown. We measured histone PTM partition to sister chromatids in embryonic stem cells. We found that parental histones H3-H4 segregate to both daughter DNA strands with a weak leading-strand bias, skewing partition at topologically associating domain (TAD) borders and enhancers proximal to replication initiation zones. Segregation of parental histones to the leading strand increased markedly in cells with histone-binding mutations in MCM2, part of the replicative helicase, exacerbating histone PTM sister chromatid asymmetry. This work reveals how histones are inherited to sister chromatids and identifies a mechanism by which the replication machinery ensures symmetric cell division. [1]: /lookup/doi/10.1126/science.aat8849 [2]: /lookup/doi/10.1126/science.aau0294 [3]: /lookup/doi/10.1126/science.aav0871</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Lloret-Llinares2018-ig" class="col-sm-10"> <div class="title">The RNA exosome contributes to gene expression regulation during stem cell differentiation</div> <div class="author"> Marta Lloret-Llinares, Evdoxia Karadoulama, Yun Chen, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Luke A Wojenski, Geno J Villafano, Jette Bornholdt, Robin Andersson, Leighton Core, Albin Sandelin, Torben Heick Jensen' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Sep 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/nar/gky817" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract. Gene expression programs change during cellular transitions. It is well established that a network of transcription factors and chromatin modifiers r</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Dalby2018-lc" class="col-sm-10"> <div class="title">FANTOM5 transcribed enhancers in mm10</div> <div class="author"> Maria Dalby, Sarah Rennie, and Robin Andersson </div> <div class="periodical"> Sep 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Overview Transcribed enhancers were identified and their expression was quantified across all human FANTOM5 libraries, following the re-aligned FANTOM5 CAGE data upon mm10 (GRCm38) (obtained from http://fantom.gsc.riken.jp/5/datafiles/reprocessed/mm10_v1/basic/), and decomposition-based peak identification (obtained from https://zenodo.org/record/545682#.WPuNy1Pyv2Q) by Kawaji, Hideya. Description Transcribed enhancers were called based on bidirectional balanced RNA signatures as per Andersson et al (2014). Enhancers were only identified distal to known exons (+/-100bp region from boundaries) and transcription start sites (+/-300bp), defined by GENCODE vM7 annotation. In total, 44,138 enhancers were identified across 1,068 libraries and the expression was quantified. For details regarding the identification of transcribed enhancers from CAGE data, please see Andersson et al (2014) and blog post. Due to varying noise levels across FANTOM5 libraries and the intrinsic low expression levels of transcribed enhancers, library-specific noise levels were estimated to define of robust set of enhancers in each sample. In summary, for each library, expression was quantified in randomly sampled genomic regions distal to assembly gaps, DNase hypersensitive sites (ENCODE), known exons and gene TSSs (GENCODE vM7) to create a genomic background expression distribution. For each library, we called an enhancer active (used) if its expression was above the 99.9th quantile of the library’s genomic background expression distribution. The robust set of enhancers consist of those significantly expressed in at least one library. While this approach ensures less permissive enhancer calling in noisy libraries, for some libraries the noise threshold is zero meaning that a single CAGE tag is sufficient for calling an enhancer active. Furthermore, the possibility of detecting enhancer transcription is affected by sequencing depth, so the number of active enhancers per library might not be biologically meaningful to compare when sequencing depths differ. Data files Each predicted enhancer is described in BED12 format with two blocks denoting the merged regions of transcription initiation on the minus and plus strands. The thickStart and thickEnd columns denote the inferred mid position between blocks of transcription initiation events. Expression and usage matrices are tab delimited and the first row gives the FANTOM5 CNhs IDs and the first column the enhancer ID (same as column 4 in BED file). Usage matrices contain zeroes and ones (0:not used, 1:used). enhancers (BED12 format) enhancer expression matrix (tab delimited, first row: CNhs IDs, first column: enhancer ID (coordinate)) binary enhancer usage matrix (0:not used, 1:used, tab delimited, first row: CNhs IDs, first column: enhancer ID (coordinate))</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Rennie2018-wl" class="col-sm-10"> <div class="title">Transcription start site analysis reveals widespread divergent transcription in D. melanogaster and core promoter-encoded enhancer activities</div> <div class="author"> Sarah Rennie, Maria Dalby, Marta Lloret-Llinares, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Stylianos Bakoulis, Christian Dalager Vaagensø, Torben Heick Jensen, Robin Andersson' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Apr 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/nar/gky244" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Mammalian gene promoters and enhancers share many properties. They are composed of a unified promoter architecture of divergent transcripton initiation and gene promoters may exhibit enhancer function. However, it is currently unclear how expression strength of a regulatory element relates to its enhancer strength and if the unifying architecture is conserved across Metazoa. Here we investigate the transcription initiation landscape and its associated RNA decay in Drosophila melanogaster. We find that the majority of active gene-distal enhancers and a considerable fraction of gene promoters are divergently transcribed. We observe quantitative relationships between enhancer potential, expression level and core promoter strength, providing an explanation for indirectly related histone modifications that are reflecting expression levels. Lowly abundant unstable RNAs initiated from weak core promoters are key characteristics of gene-distal developmental enhancers, while the housekeeping enhancer strengths of gene promoters reflect their expression strengths. The seemingly separable layer of regulation by gene promoters with housekeeping enhancer potential is also indicated by chromatin interaction data. Our results suggest a unified promoter architecture of many D. melanogaster regulatory elements, that is universal across Metazoa, whose regulatory functions seem to be related to their core promoter elements.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Baillie2018-bc" class="col-sm-10"> <div class="title">Shared activity patterns arising at genetic susceptibility loci reveal underlying genomic and cellular architecture of human disease</div> <div class="author"> J Kenneth Baillie, Andrew Bretherick, Christopher S Haley, and <span class="more-authors" title="click to view 32 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '32 more authors' ? 'Sara Clohisey, Alan Gray, Lucile P A Neyton, Jeffrey Barrett, Eli A Stahl, Albert Tenesa, Robin Andersson, J Ben Brown, Geoffrey J Faulkner, Marina Lizio, Ulf Schaefer, Carsten Daub, Masayoshi Itoh, Naoto Kondo, Timo Lassmann, Jun Kawai, IIBDGC Consortium, Damian Mole, Vladimir B Bajic, Peter Heutink, Michael Rehli, Hideya Kawaji, Albin Sandelin, Harukazu Suzuki, Jack Satsangi, Christine A Wells, Nir Hacohen, Thomas C Freeman, Yoshihide Hayashizaki, Piero Carninci, Alistair R R Forrest, David A Hume' : '32 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">32 more authors</span> </div> <div class="periodical"> <em>PLoS Comput. Biol.</em>, Mar 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pcbi.1005934" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Genetic variants underlying complex traits, including disease susceptibility, are enriched within the transcriptional regulatory elements, promoters and enhancers. There is emerging evidence that regulatory elements associated with particular traits or diseases share similar patterns of transcriptional activity. Accordingly, shared transcriptional activity (coexpression) may help prioritise loci associated with a given trait, and help to identify underlying biological processes. Using cap analysis of gene expression (CAGE) profiles of promoter- and enhancer-derived RNAs across 1824 human samples, we have analysed coexpression of RNAs originating from trait-associated regulatory regions using a novel quantitative method (network density analysis; NDA). For most traits studied, phenotype-associated variants in regulatory regions were linked to tightly-coexpressed networks that are likely to share important functional characteristics. Coexpression provides a new signal, independent of phenotype association, to enable fine mapping of causative variants. The NDA coexpression approach identifies new genetic variants associated with specific traits, including an association between the regulation of the OCT1 cation transporter and genetic variants underlying circulating cholesterol levels. NDA strongly implicates particular cell types and tissues in disease pathogenesis. For example, distinct groupings of disease-associated regulatory regions implicate two distinct biological processes in the pathogenesis of ulcerative colitis; a further two separate processes are implicated in Crohn’s disease. Thus, our functional analysis of genetic predisposition to disease defines new distinct disease endotypes. We predict that patients with a preponderance of susceptibility variants in each group are likely to respond differently to pharmacological therapy. Together, these findings enable a deeper biological understanding of the causal basis of complex traits.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Rennie2018-qp" class="col-sm-10"> <div class="title">Transcriptional decomposition reveals active chromatin architectures and cell specific regulatory interactions</div> <div class="author"> Sarah Rennie, Maria Dalby, Lucas Duin, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Robin Andersson' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Nat. Commun.</em>, Feb 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41467-017-02798-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Transcriptional regulation is tightly coupled with chromosomal positioning and three-dimensional chromatin architecture. However, it is unclear what proportion of transcriptional activity is reflecting such organisation, how much can be informed by RNA expression alone and how this impacts disease. Here, we develop a computational transcriptional decomposition approach separating the proportion of expression associated with genome organisation from independent effects not directly related to genomic positioning. We show that positionally attributable expression accounts for a considerable proportion of total levels and is highly informative of topological associating domain activities and organisation, revealing boundaries and chromatin compartments. Furthermore, expression data alone accurately predict individual enhancer-promoter interactions, drawing features from expression strength, stabilities, insulation and distance. We characterise predictions in 76 human cell types, observing extensive sharing of domains, yet highly cell-type-specific enhancer-promoter interactions and strong enrichments in relevant trait-associated variants. Overall, our work demonstrates a close relationship between transcription and chromatin architecture.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Grarup2018-gw" class="col-sm-10"> <div class="title">Loss-of-function variants in ADCY3 increase risk of obesity and type 2 diabetes</div> <div class="author"> Niels Grarup, Ida Moltke, Mette K Andersen, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Maria Dalby, Kristoffer Vitting-Seerup, Timo Kern, Yuvaraj Mahendran, Emil Jørsboe, Christina V L Larsen, Inger K Dahl-Petersen, Arthur Gilly, Daniel Suveges, George Dedoussis, Eleftheria Zeggini, Oluf Pedersen, Robin Andersson, Peter Bjerregaard, Marit E Jørgensen, Anders Albrechtsen, Torben Hansen' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>Nat. Genet.</em>, Feb 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41588-017-0022-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We have identified a variant in ADCY3 (encoding adenylate cyclase 3) associated with markedly increased risk of obesity and type 2 diabetes in the Greenlandic population. The variant disrupts a splice acceptor site, and carriers have decreased ADCY3 RNA expression. Additionally, we observe an enrichment of rare ADCY3 loss-of-function variants among individuals with type 2 diabetes in trans-ancestry cohorts. These findings provide new information on disease etiology relevant for future treatment strategies.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Boyd2018-rc" class="col-sm-10"> <div class="title">Characterization of the enhancer and promoter landscape of inflammatory bowel disease from human colon biopsies</div> <div class="author"> Mette Boyd, Malte Thodberg, Morana Vitezic, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Jette Bornholdt, Kristoffer Vitting-Seerup, Yun Chen, Mehmet Coskun, Yuan Li, Bobby Zhao Sheng Lo, Pia Klausen, Pawel Jan Schweiger, Anders Gorm Pedersen, Nicolas Rapin, Kerstin Skovgaard, Katja Dahlgaard, Robin Andersson, Thilde Bagger Terkelsen, Berit Lilje, Jesper Thorvald Troelsen, Andreas Munk Petersen, Kim Bak Jensen, Ismail Gögenur, Peter Thielsen, Jakob Benedict Seidelin, Ole Haagen Nielsen, Jacob Tveiten Bjerrum, Albin Sandelin' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Nat. Commun.</em>, Jan 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41467-018-03766-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Inflammatory bowel disease (IBD) is a chronic intestinal disorder, with two main types: Crohn’s disease (CD) and ulcerative colitis (UC), whose molecular pathology is not well understood. The majority of IBD-associated SNPs are located in non-coding regions and are hard to characterize since regulatory regions in IBD are not known. Here we profile transcription start sites (TSSs) and enhancers in the descending colon of 94 IBD patients and controls. IBD-upregulated promoters and enhancers are highly enriched for IBD-associated SNPs and are bound by the same transcription factors. IBD-specific TSSs are associated to genes with roles in both inflammatory cascades and gut epithelia while TSSs distinguishing UC and CD are associated to gut epithelia functions. We find that as few as 35 TSSs can distinguish active CD, UC, and controls with 85% accuracy in an independent cohort. Our data constitute a foundation for understanding the molecular pathology, gene regulation, and genetics of IBD.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Bornholdt2017-wc" class="col-sm-10"> <div class="title">Identification of Gene Transcription Start Sites and Enhancers Responding to Pulmonary Carbon Nanotube Exposure in Vivo</div> <div class="author"> Jette Bornholdt, Anne Thoustrup Saber, Berit Lilje, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Mette Boyd, Mette Jørgensen, Yun Chen, Morana Vitezic, Nicklas Raun Jacobsen, Sarah Søs Poulsen, Trine Berthing, Simon Bressendorff, Kristoffer Vitting-Seerup, Robin Andersson, Karin Sørig Hougaard, Carole L Yauk, Sabina Halappanavar, Håkan Wallin, Ulla Vogel, Albin Sandelin' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>ACS Nano</em>, Apr 2017 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1021/acsnano.6b07533" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Lizio2017-yd" class="col-sm-10"> <div class="title">Update of the FANTOM web resource: high resolution transcriptome of diverse cell types in mammals</div> <div class="author"> Marina Lizio, Jayson Harshbarger, Imad Abugessaisa, and <span class="more-authors" title="click to view 22 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '22 more authors' ? 'Shuei Noguchi, Atsushi Kondo, Jessica Severin, Chris Mungall, David Arenillas, Anthony Mathelier, Yulia A Medvedeva, Andreas Lennartsson, Finn Drablos, Jordan A Ramilowski, Owen Rackham, Julian Gough, Robin Andersson, Albin Sandelin, Hans Ienasescu, Hiromasa Ono, Hidemasa Bono, Yoshihide Hayashizaki, Piero Carninci, Alistair R R Forrest, Takeya Kasukawa, Hideya Kawaji' : '22 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">22 more authors</span> </div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Jan 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/nar/gkw995" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Upon the first publication of the fifth iteration of the Functional Annotation of Mammalian Genomes collaborative project, FANTOM5, we gathered a series of primary data and database systems into the FANTOM web resource (http://fantom.gsc.riken.jp) to facilitate researchers to explore transcriptional regulation and cellular states. In the course of the collaboration, primary data and analysis results have been expanded, and functionalities of the database systems enhanced. We believe that our data and web systems are invaluable resources, and we think the scientific community will benefit for this recent update to deepen their understanding of mammalian cellular organization. We introduce the contents of FANTOM5 here, report recent updates in the web resource and provide future perspectives.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Ehrlund2017-mw" class="col-sm-10"> <div class="title">Transcriptional Dynamics During Human Adipogenesis and Its Link to Adipose Morphology and Distribution</div> <div class="author"> Anna Ehrlund, Niklas Mejhert, Christel Björk, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Robin Andersson, Agné Kulyté, Gaby Åström, Masayoshi Itoh, Hideya Kawaji, Timo Lassmann, Carsten O Daub, Piero Carninci, Alistair R R Forrest, Yoshihide Hayashizaki, Albin Sandelin, Erik Ingelsson, FANTOM Consortium, Mikael Rydén, Jurga Laurencikiene, Peter Arner, Erik Arner' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>Diabetes</em>, Jan 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.2337/db16-0631" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>White adipose tissue (WAT) can develop into several phenotypes with different pathophysiological impact on type 2 diabetes. To better understand the adipogenic process, the transcriptional events that occur during in vitro differentiation of human adipocytes were investigated and the findings linked to WAT phenotypes. Single-molecule transcriptional profiling provided a detailed map of the expressional changes of genes, enhancers, and long noncoding RNAs, where different types of transcripts share common dynamics during differentiation. Common signatures include early downregulated, transient, and late induced transcripts, all of which are linked to distinct developmental processes during adipogenesis. Enhancers expressed during adipogenesis overlap significantly with genetic variants associated with WAT distribution. Transiently expressed and late induced genes are associated with hypertrophic WAT (few but large fat cells), a phenotype closely linked to insulin resistance and type 2 diabetes. Transcription factors that are expressed early or transiently affect differentiation and adipocyte function and are controlled by several well-known upstream regulators such as glucocorticosteroids, insulin, cAMP, and thyroid hormones. Taken together, our results suggest a complex but highly coordinated regulation of adipogenesis.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Chen2016-rk" class="col-sm-10"> <div class="title">Principles for RNA metabolism and alternative transcription initiation within closely spaced promoters</div> <div class="author"> Yun Chen, Athma A Pai, Jan Herudek, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Michal Lubas, Nicola Meola, Aino I Järvelin, Robin Andersson, Vicent Pelechano, Lars M Steinmetz, Torben Heick Jensen, Albin Sandelin' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Nat. Genet.</em>, Sep 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/ng.3616" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Mammalian transcriptomes are complex and formed by extensive promoter activity. In addition, gene promoters are largely divergent and initiate transcription of reverse-oriented promoter upstream transcripts (PROMPTs). Although PROMPTs are commonly terminated early, influenced by polyadenylation sites, promoters often cluster so that the divergent activity of one might impact another. Here we found that the distance between promoters strongly correlates with the expression, stability and length of their associated PROMPTs. Adjacent promoters driving divergent mRNA transcription support PROMPT formation, but owing to polyadenylation site constraints, these transcripts tend to spread into the neighboring mRNA on the same strand. This mechanism to derive new alternative mRNA transcription start sites (TSSs) is also evident at closely spaced promoters supporting convergent mRNA transcription. We suggest that basic building blocks of divergently transcribed core promoter pairs, in combination with the wealth of TSSs in mammalian genomes, provide a framework with which evolution shapes transcriptomes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Persson2016-ff" class="col-sm-10"> <div class="title">Regulating retrotransposon activity through the use of alternative transcription start sites</div> <div class="author"> Jenna Persson, Babett Steglich, Agata Smialowska, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Mette Boyd, Jette Bornholdt, Robin Andersson, Catherine Schurra, Benoit Arcangioli, Albin Sandelin, Olaf Nielsen, Karl Ekwall' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>EMBO Rep.</em>, Feb 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.15252/embr.201541866" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Retrotransposons, the ancestors of retroviruses, have the potential for gene disruption and genomic takeover if not kept in check. Paradoxically, although host cells repress these elements by multiple mechanisms, they are transcribed and are even activated under stress conditions. Here, we describe a new mechanism of retrotransposon regulation through transcription start site (TSS) selection by altered nucleosome occupancy. We show that Fun30 chromatin remodelers cooperate to maintain a high level of nucleosome occupancy at retrotransposon-flanking long terminal repeat (LTR) elements. This enforces the use of a downstream TSS and the production of a truncated RNA incapable of reverse transcription and retrotransposition. However, in stressed cells, nucleosome occupancy at LTR elements is reduced, and the TSS shifts to allow for productive transcription. We propose that controlled retrotransposon transcription from a nonproductive TSS allows for rapid stress-induced activation, while preventing uncontrolled transposon activity in the genome.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Kaczkowski2016-ps" class="col-sm-10"> <div class="title">Transcriptome Analysis of Recurrently Deregulated Genes across Multiple Cancers Identifies New Pan-Cancer Biomarkers</div> <div class="author"> Bogumil Kaczkowski, Yuji Tanaka, Hideya Kawaji, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Albin Sandelin, Robin Andersson, Masayoshi Itoh, Timo Lassmann, Yoshihide Hayashizaki, Piero Carninci, Alistair R R Forrest, FANTOM5 consortium' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Cancer Res.</em>, Jan 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1158/0008-5472.CAN-15-0484" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Genes that are commonly deregulated in cancer are clinically attractive as candidate pan-diagnostic markers and therapeutic targets. To globally identify such targets, we compared Cap Analysis of Gene Expression profiles from 225 different cancer cell lines and 339 corresponding primary cell samples to identify transcripts that are deregulated recurrently in a broad range of cancer types. Comparing RNA-seq data from 4,055 tumors and 563 normal tissues profiled in the The Cancer Genome Atlas and FANTOM5 datasets, we identified a core transcript set with theranostic potential. Our analyses also revealed enhancer RNAs, which are upregulated in cancer, defining promoters that overlap with repetitive elements (especially SINE/Alu and LTR/ERV1 elements) that are often upregulated in cancer. Lastly, we documented for the first time upregulation of multiple copies of the REP522 interspersed repeat in cancer. Overall, our genome-wide expression profiling approach identified a comprehensive set of candidate biomarkers with pan-cancer potential, and extended the perspective and pathogenic significance of repetitive elements that are frequently activated during cancer progression. Cancer Res; 76(2); 216-26. ©2015 AACR.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Ienasescu2016-cl" class="col-sm-10"> <div class="title">On-the-fly selection of cell-specific enhancers, genes, miRNAs and proteins across the human body using SlideBase</div> <div class="author"> Hans Ienasescu, Kang Li, Robin Andersson, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Morana Vitezic, Sarah Rennie, Yun Chen, Kristoffer Vitting-Seerup, Emil Lagoni, Mette Boyd, Jette Bornholdt, Michiel J L Hoon, Hideya Kawaji, Timo Lassmann, FANTOM Consortium, Yoshihide Hayashizaki, Alistair R R Forrest, Piero Carninci, Albin Sandelin' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Database</em>, Jan 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/database/baw144" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Genomics consortia have produced large datasets profiling the expression of genes, micro-RNAs, enhancers and more across human tissues or cells. There is a need for intuitive tools to select subsets of such data that is the most relevant for specific studies. To this end, we present SlideBase, a web tool which offers a new way of selecting genes, promoters, enhancers and microRNAs that are preferentially expressed/used in a specified set of cells/tissues, based on the use of interactive sliders. With the help of sliders, SlideBase enables users to define custom expression thresholds for individual cell types/tissues, producing sets of genes, enhancers etc. which satisfy these constraints. Changes in slider settings result in simultaneous changes in the selected sets, updated in real time. SlideBase is linked to major databases from genomics consortia, including FANTOM, GTEx, The Human Protein Atlas and BioGPS.Database URL: http://slidebase.binf.ku.dk.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Andersson2015-hn" class="col-sm-10"> <div class="title">Human Gene Promoters Are Intrinsically Bidirectional</div> <div class="author"> Robin Andersson, Yun Chen, Leighton Core, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'John T Lis, Albin Sandelin, Torben Heick Jensen' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Mol. Cell</em>, Nov 2015 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1016/j.molcel.2015.10.015" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Young2015-xu" class="col-sm-10"> <div class="title">The frequent evolutionary birth and death of functional promoters in mouse and human</div> <div class="author"> Robert S Young, Yoshihide Hayashizaki, Robin Andersson, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Albin Sandelin, Hideya Kawaji, Masayoshi Itoh, Timo Lassmann, Piero Carninci, The FANTOM Consortium, Wendy A Bickmore, Alistair R Forrest, Martin S Taylor' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Genome Res.</em>, "1 " # "oct" 2015 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1101/gr.190546.115" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Andersson2015-av" class="col-sm-10"> <div class="title">A unified architecture of transcriptional regulatory elements</div> <div class="author"> Robin Andersson, Albin Sandelin, and Charles G Danko </div> <div class="periodical"> <em>Trends Genet.</em>, Aug 2015 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1016/j.tig.2015.05.007" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Andersson2015-wr" class="col-sm-10"> <div class="title">Promoter or enhancer, what’s the difference? Deconstruction of established distinctions and presentation of a unifying model</div> <div class="author"> Robin Andersson </div> <div class="periodical"> <em>Bioessays</em>, Mar 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/bies.201400162" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Gene transcription is strictly controlled by the interplay of regulatory events at gene promoters and gene-distal regulatory elements called enhancers. Despite extensive studies of enhancers, we still have a very limited understanding of their mechanisms of action and their restricted spatio-temporal activities. A better understanding would ultimately lead to fundamental insights into the control of gene transcription and the action of regulatory genetic variants involved in disease. Here, I review and discuss pros and cons of state-of-the-art genomics methods to localize and infer the activity of enhancers. Among the different approaches, profiling of enhancer RNAs yields the highest specificity and may be superior in detecting in vivo activity. I discuss their apparent similarities to promoters, which challenge the established view of enhancers and promoters as distinct entities, and present a unifying model of regulatory elements in transcriptional regulation, in which activity, transcriptional output and regulatory function is context specific.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Arner2015-dw" class="col-sm-10"> <div class="title">Transcribed enhancers lead waves of coordinated transcription in transitioning mammalian cells</div> <div class="author"> Erik Arner, Carsten O Daub, Kristoffer Vitting-Seerup, and <span class="more-authors" title="click to view 107 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '107 more authors' ? 'Robin Andersson, Berit Lilje, Finn Drablos, Andreas Lennartsson, Michelle Rönnerblad, Olga Hrydziuszko, Morana Vitezic, Tom C Freeman, Ahmad M N Alhendi, Peter Arner, Richard Axton, J Kenneth Baillie, Anthony Beckhouse, Beatrice Bodega, James Briggs, Frank Brombacher, Margaret Davis, Michael Detmar, Anna Ehrlund, Mitsuhiro Endoh, Afsaneh Eslami, Michela Fagiolini, Lynsey Fairbairn, Geoffrey J Faulkner, Carmelo Ferrai, Malcolm E Fisher, Lesley Forrester, Daniel Goldowitz, Reto Guler, Thomas Ha, Mitsuko Hara, Meenhard Herlyn, Tomokatsu Ikawa, Chieko Kai, Hiroshi Kawamoto, Levon M Khachigian, S Peter Klinken, Soichi Kojima, Haruhiko Koseki, Sarah Klein, Niklas Mejhert, Ken Miyaguchi, Yosuke Mizuno, Mitsuru Morimoto, Kelly J Morris, Christine Mummery, Yutaka Nakachi, Soichi Ogishima, Mariko Okada-Hatakeyama, Yasushi Okazaki, Valerio Orlando, Dmitry Ovchinnikov, Robert Passier, Margaret Patrikakis, Ana Pombo, Xian-Yang Qin, Sugata Roy, Hiroki Sato, Suzana Savvi, Alka Saxena, Anita Schwegmann, Daisuke Sugiyama, Rolf Swoboda, Hiroshi Tanaka, Andru Tomoiu, Louise N Winteringham, Ernst Wolvetang, Chiyo Yanagi-Mizuochi, Misako Yoneda, Susan Zabierowski, Peter Zhang, Imad Abugessaisa, Nicolas Bertin, Alexander D Diehl, Shiro Fukuda, Masaaki Furuno, Jayson Harshbarger, Akira Hasegawa, Fumi Hori, Sachi Ishikawa-Kato, Yuri Ishizu, Masayoshi Itoh, Tsugumi Kawashima, Miki Kojima, Naoto Kondo, Marina Lizio, Terrence F Meehan, Christopher J Mungall, Mitsuyoshi Murata, Hiromi Nishiyori-Sueki, Serkan Sahin, Sayaka Nagao-Sato, Jessica Severin, Michiel J L Hoon, Jun Kawai, Takeya Kasukawa, Timo Lassmann, Harukazu Suzuki, Hideya Kawaji, Kim M Summers, Christine Wells, FANTOM Consortium, David A Hume, Alistair R R Forrest, Albin Sandelin, Piero Carninci, Yoshihide Hayashizaki' : '107 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">107 more authors</span> </div> <div class="periodical"> <em>Science</em>, Feb 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.1259418" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Although it is generally accepted that cellular differentiation requires changes to transcriptional networks, dynamic regulation of promoters and enhancers at specific sets of genes has not been previously studied en masse. Exploiting the fact that active promoters and enhancers are transcribed, we simultaneously measured their activity in 19 human and 14 mouse time courses covering a wide range of cell types and biological stimuli. Enhancer RNAs, then messenger RNAs encoding transcription factors, dominated the earliest responses. Binding sites for key lineage transcription factors were simultaneously overrepresented in enhancers and promoters active in each cellular system. Our data support a highly generalizable model in which enhancer transcription is the earliest event in successive waves of transcriptional change during cellular differentiation or activation.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Lennartsson2015-op" class="col-sm-10"> <div class="title">Remodeling of retrotransposon elements during epigenetic induction of adult visual cortical plasticity by HDAC inhibitors</div> <div class="author"> Andreas Lennartsson, Erik Arner, Michela Fagiolini, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Alka Saxena, Robin Andersson, Hazuki Takahashi, Yukihiko Noro, Judy Sng, Albin Sandelin, Takao K Hensch, Piero Carninci' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Epigenetics Chromatin</em>, Jan 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1186/s13072-015-0043-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>BACKGROUND:The capacity for plasticity in the adult brain is limited by the anatomical traces laid down during early postnatal life. Removing certain molecular brakes, such as histone deacetylases (HDACs), has proven to be effective in recapitulating juvenile plasticity in the mature visual cortex (V1). We investigated the chromatin structure and transcriptional control by genome-wide sequencing of DNase I hypersensitive sites (DHSS) and cap analysis of gene expression (CAGE) libraries after HDAC inhibition by valproic acid (VPA) in adult V1. RESULTS:We found that VPA reliably reactivates the critical period plasticity and induces a dramatic change of chromatin organization in V1 yielding significantly greater accessibility distant from promoters, including at enhancer regions. VPA also induces nucleosome eviction specifically from retrotransposon (in particular SINE) elements. The transiently accessible SINE elements overlap with transcription factor-binding sites of the Fox family. Mapping of transcription start site activity using CAGE revealed transcription of epigenetic and neural plasticity-regulating genes following VPA treatment, which may help to re-program the genomic landscape and reactivate plasticity in the adult cortex. CONCLUSIONS:Treatment with HDAC inhibitors increases accessibility to enhancers and repetitive elements underlying brain-specific gene expression and reactivation of visual cortical plasticity.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Boyd2014-kc" class="col-sm-10"> <div class="title">Identification of TNF-α-responsive promoters and enhancers in the intestinal epithelial cell model Caco-2</div> <div class="author"> Mette Boyd, Mehmet Coskun, Berit Lilje, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Robin Andersson, Ilka Hoof, Jette Bornholdt, Katja Dahlgaard, Jørgen Olsen, Morana Vitezic, Jacob Tveiten Bjerrum, Jakob Benedict Seidelin, Ole Haagen Nielsen, Jesper Thorvald Troelsen, Albin Sandelin' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>DNA Res.</em>, Dec 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/dnares/dsu022" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The Caco-2 cell line is one of the most important in vitro models for enterocytes, and is used to study drug absorption and disease, including inflammatory bowel disease and cancer. In order to use the model optimally, it is necessary to map its functional entities. In this study, we have generated genome-wide maps of active transcription start sites (TSSs), and active enhancers in Caco-2 cells with or without tumour necrosis factor (TNF)-α stimulation to mimic an inflammatory state. We found 520 promoters that significantly changed their usage level upon TNF-α stimulation; of these, 52% are not annotated. A subset of these has the potential to confer change in protein function due to protein domain exclusion. Moreover, we locate 890 transcribed enhancer candidates, where ∼50% are changing in usage after TNF-α stimulation. These enhancers share motif enrichments with similarly responding gene promoters. As a case example, we characterize an enhancer regulating the laminin-5 γ2-chain (LAMC2) gene by nuclear factor (NF)-κB binding. This report is the first to present comprehensive TSS and enhancer maps over Caco-2 cells, and highlights many novel inflammation-specific promoters and enhancers.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Andersson2014-yr" class="col-sm-10"> <div class="title">Nuclear stability and transcriptional directionality separate functionally distinct RNA species</div> <div class="author"> Robin Andersson, Peter Refsing Andersen, Eivind Valen, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Leighton J Core, Jette Bornholdt, Mette Boyd, Torben Heick Jensen, Albin Sandelin' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Nat. Commun.</em>, Jul 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/ncomms6336" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Enroth2014-hc" class="col-sm-10"> <div class="title">Nucleosome regulatory dynamics in response to TGFβ</div> <div class="author"> Stefan Enroth, Robin Andersson, Madhusudhan Bysani, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Ola Wallerman, Stefan Termén, Brian B Tuch, Francisco M De La Vega, Carl-Henrik Heldin, Aristidis Moustakas, Jan Komorowski, Claes Wadelius' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Jun 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/nar/gku326" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Nucleosomes play important roles in a cell beyond their basal functionality in chromatin compaction. Their placement affects all steps in transcriptional regulation, from transcription factor (TF) binding to messenger ribonucleic acid (mRNA) synthesis. Careful profiling of their locations and dynamics in response to stimuli is important to further our understanding of transcriptional regulation by the state of chromatin. We measured nucleosome occupancy in human hepatic cells before and after treatment with transforming growth factor beta 1 (TGFβ1), using massively parallel sequencing. With a newly developed method, SuMMIt, for precise positioning of nucleosomes we inferred dynamics of the nucleosomal landscape. Distinct nucleosome positioning has previously been described at transcription start site and flanking TF binding sites. We found that the average pattern is present at very few sites and, in case of TF binding, the double peak surrounding the sites is just an artifact of averaging over many loci. We systematically searched for depleted nucleosomes in stimulated cells compared to unstimulated cells and identified 24 318 loci. Depending on genomic annotation, 44-78% of them were over-represented in binding motifs for TFs. Changes in binding affinity were verified for HNF4α by qPCR. Strikingly many of these loci were associated with expression changes, as measured by RNA sequencing.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Davis2014-hi" class="col-sm-10"> <div class="title">Transcriptional profiling of the human fibrillin/LTBP gene family, key regulators of mesenchymal cell functions</div> <div class="author"> Margaret R Davis, Robin Andersson, Jessica Severin, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Michiel Hoon, Nicolas Bertin, J Kenneth Baillie, Hideya Kawaji, Albin Sandelin, Alistair R R Forrest, Kim M Summers' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Mol. Genet. Metab.</em>, May 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1016/j.ymgme.2013.12.006" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Fort2014-gi" class="col-sm-10"> <div class="title">Deep transcriptome profiling of mammalian stem cells supports a regulatory role for retrotransposons in pluripotency maintenance</div> <div class="author"> Alexandre Fort, Kosuke Hashimoto, Daisuke Yamada, and <span class="more-authors" title="click to view 18 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '18 more authors' ? ' Salimullah, Chaman A Keya, Alka Saxena, Alessandro Bonetti, Irina Voineagu, Nicolas Bertin, Anton Kratz, Yukihiko Noro, Chee-Hong Wong, Michiel Hoon, Robin Andersson, Albin Sandelin, Harukazu Suzuki, Chia-Lin Wei, Haruhiko Koseki, Yuki Hasegawa, Alistair R R Forrest, Piero Carninci' : '18 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">18 more authors</span> </div> <div class="periodical"> <em>Nat. Genet.</em>, Apr 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/ng.2965" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Morikawa2014-zh" class="col-sm-10"> <div class="title">Differential roles of epigenetic changes and Foxp3 expression in regulatory T cell-specific transcriptional regulation</div> <div class="author"> Hiromasa Morikawa, Naganari Ohkura, Alexis Vandenbon, and <span class="more-authors" title="click to view 270 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '270 more authors' ? 'Masayoshi Itoh, Sayaka Nagao-Sato, Hideya Kawaji, Timo Lassmann, Piero Carninci, Yoshihide Hayashizaki, Alistair R R Forrest, Daron M Standley, Hiroshi Date, Shimon Sakaguchi, the FANTOM Consortium, Alistair R R Forrest, Hideya Kawaji, Michael Rehli, J Kenneth Baillie, Michiel J L Hoon, Vanja Haberle, Timo Lassmann, Ivan V Kulakovskiy, Marina Lizio, Masayoshi Itoh, Robin Andersson, Christopher J Mungall, Terrence F Meehan, Sebastian Schmeier, Nicolas Bertin, Mette Jørgensen, Emmanuel Dimont, Erik Arner, Christian Schmidl, Ulf Schaefer, Yulia A Medvedeva, Charles Plessy, Morana Vitezic, Jessica Severin, Colin A Semple, Yuri Ishizu, Margherita Francescatto, Intikhab Alam, Davide Albanese, Gabriel M Altschuler, John A C Archer, Peter Arner, Magda Babina, Sarah Baker, Piotr J Balwierz, Anthony G Beckhouse, Swati Pradhan-Bhatt, Judith A Blake, Antje Blumenthal, Beatrice Bodega, Alessandro Bonetti, James Briggs, Frank Brombacher, A Maxwell Burroughs, Andrea Califano, Carlo V Cannistraci, Daniel Carbajo, Yun Chen, Marco Chierici, Yari Ciani, Hans C Clevers, Emiliano Dalla, Carrie A Davis, Bart Deplancke, Michael Detmar, Alexander D Diehl, Taeko Dohi, Finn Drabløs, Albert S B Edge, Matthias Edinger, Karl Ekwall, Mitsuhiro Endoh, Hideki Enomoto, Michela Fagiolini, Lynsey Fairbairn, Hai Fang, Mary C Farach-Carson, Geoffrey J Faulkner, Alexander V Favorov, Malcolm E Fisher, Martin C Frith, Rie Fujita, Shiro Fukuda, Cesare Furlanello, Masaaki Furuno, Jun-Ichi Furusawa, Teunis B Geijtenbeek, Andrew Gibson, Thomas Gingeras, Daniel Goldowitz, Julian Gough, Sven Guhl, Reto Guler, Stefano Gustincich, Thomas J Ha, Masahide Hamaguchi, Mitsuko Hara, Matthias Harbers, Jayson Harshbarger, Akira Hasegawa, Yuki Hasegawa, Takehiro Hashimoto, Meenhard Herlyn, Kelly J Hitchens, Shannan J Ho Sui, Oliver M Hofmann, Ilka Hoof, Fumi Hori, Lukasz Huminiecki, Kei Iida, Tomokatsu Ikawa, Boris R Jankovic, Hui Jia, Anagha Joshi, Giuseppe Jurman, Bogumil Kaczkowski, Chieko Kai, Kaoru Kaida, Ai Kaiho, Kazuhiro Kajiyama, Mutsumi Kanamori-Katayama, Artem S Kasianov, Takeya Kasukawa, Shintaro Katayama, Sachi Kato, Shuji Kawaguchi, Hiroshi Kawamoto, Yuki I Kawamura, Tsugumi Kawashima, Judith S Kempfle, Tony J Kenna, Juha Kere, Levon M Khachigian, Toshio Kitamura, S Peter Klinken, Alan J Knox, Miki Kojima, Soichi Kojima, Naoto Kondo, Haruhiko Koseki, Shigeo Koyasu, Sarah Krampitz, Atsutaka Kubosaki, Andrew T Kwon, Jeroen F J Laros, Weonju Lee, Andreas Lennartsson, Kang Li, Berit Lilje, Leonard Lipovich, Alan Mackay-sim, Ri-Ichiroh Manabe, Jessica C Mar, Benoit Marchand, Anthony Mathelier, Niklas Mejhert, Alison Meynert, Yosuke Mizuno, David A de Lima Morais, Hiromasa Morikawa, Mitsuru Morimoto, Kazuyo Moro, Efthymios Motakis, Hozumi Motohashi, Christine L Mummery, Mitsuyoshi Murata, Sayaka Nagao-Sato, Yutaka Nakachi, Fumio Nakahara, Toshiyuki Nakamura, Yukio Nakamura, Kenichi Nakazato, Erik Nimwegen, Noriko Ninomiya, Hiromi Nishiyori, Shohei Noma, Tadasuke Nozaki, Soichi Ogishima, Naganari Ohkura, Hiroko Ohmiya, Hiroshi Ohno, Mitsuhiro Ohshima, Mariko Okada-Hatakeyama, Yasushi Okazaki, Valerio Orlando, Dmitry A Ovchinnikov, Arnab Pain, Robert Passier, Margaret Patrikakis, Helena Persson, Silvano Piazza, James G D Prendergast, Owen J L Rackham, Jordan A Ramilowski, Mamoon Rashid, Timothy Ravasi, Patrizia Rizzu, Marco Roncador, Sugata Roy, Morten B Rye, Eri Saijyo, Antti Sajantila, Akiko Saka, Shimon Sakaguchi, Mizuho Sakai, Hiroki Sato, Hironori Satoh, Suzana Savvi, Alka Saxena, Claudio Schneider, Erik A Schultes, Gundula G Schulze-Tanzil, Anita Schwegmann, Thierry Sengstag, Guojun Sheng, Hisashi Shimoji, Yishai Shimoni, Jay W Shin, Christophe Simon, Daisuke Sugiyama, Takaaki Sugiyama, Masanori Suzuki, Rolf K Swoboda, Peter A C Hoen, Michihira Tagami, Naoko Takahashi, Jun Takai, Hiroshi Tanaka, Hideki Tatsukawa, Zuotian Tatum, Mark Thompson, Hiroo Toyoda, Tetsuro Toyoda, Eivind Valen, Marc Wetering, Linda M Berg, Roberto Verardo, Dipti Vijayan, Ilya E Vorontsov, Wyeth W Wasserman, Shoko Watanabe, Christine A Wells, Louise N Winteringham, Ernst Wolvetang, Emily J Wood, Yoko Yamaguchi, Masayuki Yamamoto, Misako Yoneda, Yohei Yonekura, Shigehiro Yoshida, Suzan E Zabierowski, Peter G Zhang, Xiaobei Zhao, Silvia Zucchelli, Kim M Summers, Harukazu Suzuki, Carsten O Daub, Jun Kawai, Peter Heutink, Winston Hide, Tom C Freeman, Boris Lenhard, Vladimir B Bajic, Martin S Taylor, Vsevolod J Makeev, Albin Sandelin, David A Hume, Piero Carninci, Yoshihide Hayashizaki' : '270 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">270 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, Apr 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1073/pnas.1312717110" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Naturally occurring regulatory T (Treg) cells, which specifically express the transcription factor forkhead box P3 (Foxp3), are engaged in the maintenance of immunological self-tolerance and homeostasis. By transcriptional start site cluster analysis, we assessed here how genome-wide patterns of DNA methylation or Foxp3 binding sites were associated with Treg-specific gene expression. We found that Treg-specific DNA hypomethylated regions were closely associated with Treg up-regulated transcriptional start site clusters, whereas Foxp3 binding regions had no significant correlation with either up- or down-regulated clusters in nonactivated Treg cells. However, in activated Treg cells, Foxp3 binding regions showed a strong correlation with down-regulated clusters. In accordance with these findings, the above two features of activation-dependent gene regulation in Treg cells tend to occur at different locations in the genome. The results collectively indicate that Treg-specific DNA hypomethylation is instrumental in gene up-regulation in steady state Treg cells, whereas Foxp3 down-regulates the expression of its target genes in activated Treg cells. Thus, the two events seem to play distinct but complementary roles in Treg-specific gene expression.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="FANTOM-Consortium-and-the-RIKEN-PMI-and-CLST-DGT-2014-ra" class="col-sm-10"> <div class="title">A promoter-level mammalian expression atlas</div> <div class="author"> FANTOM Consortium and the RIKEN PMI and CLST (DGT), Alistair R R Forrest, Hideya Kawaji, and <span class="more-authors" title="click to view 259 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '259 more authors' ? 'Michael Rehli, J Kenneth Baillie, Michiel J L Hoon, Vanja Haberle, Timo Lassmann, Ivan V Kulakovskiy, Marina Lizio, Masayoshi Itoh, Robin Andersson, Christopher J Mungall, Terrence F Meehan, Sebastian Schmeier, Nicolas Bertin, Mette Jørgensen, Emmanuel Dimont, Erik Arner, Christian Schmidl, Ulf Schaefer, Yulia A Medvedeva, Charles Plessy, Morana Vitezic, Jessica Severin, Colin A Semple, Yuri Ishizu, Robert S Young, Margherita Francescatto, Intikhab Alam, Davide Albanese, Gabriel M Altschuler, Takahiro Arakawa, John A C Archer, Peter Arner, Magda Babina, Sarah Rennie, Piotr J Balwierz, Anthony G Beckhouse, Swati Pradhan-Bhatt, Judith A Blake, Antje Blumenthal, Beatrice Bodega, Alessandro Bonetti, James Briggs, Frank Brombacher, A Maxwell Burroughs, Andrea Califano, Carlo V Cannistraci, Daniel Carbajo, Yun Chen, Marco Chierici, Yari Ciani, Hans C Clevers, Emiliano Dalla, Carrie A Davis, Michael Detmar, Alexander D Diehl, Taeko Dohi, Finn Drabløs, Albert S B Edge, Matthias Edinger, Karl Ekwall, Mitsuhiro Endoh, Hideki Enomoto, Michela Fagiolini, Lynsey Fairbairn, Hai Fang, Mary C Farach-Carson, Geoffrey J Faulkner, Alexander V Favorov, Malcolm E Fisher, Martin C Frith, Rie Fujita, Shiro Fukuda, Cesare Furlanello, Masaaki Furino, Jun-Ichi Furusawa, Teunis B Geijtenbeek, Andrew P Gibson, Thomas Gingeras, Daniel Goldowitz, Julian Gough, Sven Guhl, Reto Guler, Stefano Gustincich, Thomas J Ha, Masahide Hamaguchi, Mitsuko Hara, Matthias Harbers, Jayson Harshbarger, Akira Hasegawa, Yuki Hasegawa, Takehiro Hashimoto, Meenhard Herlyn, Kelly J Hitchens, Shannan J Ho Sui, Oliver M Hofmann, Ilka Hoof, Furni Hori, Lukasz Huminiecki, Kei Iida, Tomokatsu Ikawa, Boris R Jankovic, Hui Jia, Anagha Joshi, Giuseppe Jurman, Bogumil Kaczkowski, Chieko Kai, Kaoru Kaida, Ai Kaiho, Kazuhiro Kajiyama, Mutsumi Kanamori-Katayama, Artem S Kasianov, Takeya Kasukawa, Shintaro Katayama, Sachi Kato, Shuji Kawaguchi, Hiroshi Kawamoto, Yuki I Kawamura, Tsugumi Kawashima, Judith S Kempfle, Tony J Kenna, Juha Kere, Levon M Khachigian, Toshio Kitamura, S Peter Klinken, Alan J Knox, Miki Kojima, Soichi Kojima, Naoto Kondo, Haruhiko Koseki, Shigeo Koyasu, Sarah Krampitz, Atsutaka Kubosaki, Andrew T Kwon, Jeroen F J Laros, Weonju Lee, Andreas Lennartsson, Kang Li, Berit Lilje, Leonard Lipovich, Alan Mackay-Sim, Ri-Ichiroh Manabe, Jessica C Mar, Benoit Marchand, Anthony Mathelier, Niklas Mejhert, Alison Meynert, Yosuke Mizuno, David A Lima Morais, Hiromasa Morikawa, Mitsuru Morimoto, Kazuyo Moro, Efthymios Motakis, Hozumi Motohashi, Christine L Mummery, Mitsuyoshi Murata, Sayaka Nagao-Sato, Yutaka Nakachi, Fumio Nakahara, Toshiyuki Nakamura, Yukio Nakamura, Kenichi Nakazato, Erik Nimwegen, Noriko Ninomiya, Hiromi Nishiyori, Shohei Noma, Shohei Noma, Tadasuke Noazaki, Soichi Ogishima, Naganari Ohkura, Hiroko Ohimiya, Hiroshi Ohno, Mitsuhiro Ohshima, Mariko Okada-Hatakeyama, Yasushi Okazaki, Valerio Orlando, Dmitry A Ovchinnikov, Arnab Pain, Robert Passier, Margaret Patrikakis, Helena Persson, Silvano Piazza, James G D Prendergast, Owen J L Rackham, Jordan A Ramilowski, Mamoon Rashid, Timothy Ravasi, Patrizia Rizzu, Marco Roncador, Sugata Roy, Morten B Rye, Eri Saijyo, Antti Sajantila, Akiko Saka, Shimon Sakaguchi, Mizuho Sakai, Hiroki Sato, Suzana Savvi, Alka Saxena, Claudio Schneider, Erik A Schultes, Gundula G Schulze-Tanzil, Anita Schwegmann, Thierry Sengstag, Guojun Sheng, Hisashi Shimoji, Yishai Shimoni, Jay W Shin, Christophe Simon, Daisuke Sugiyama, Takaai Sugiyama, Masanori Suzuki, Naoko Suzuki, Rolf K Swoboda, Peter A C Hoen, Michihira Tagami, Naoko Takahashi, Jun Takai, Hiroshi Tanaka, Hideki Tatsukawa, Zuotian Tatum, Mark Thompson, Hiroo Toyodo, Tetsuro Toyoda, Elvind Valen, Marc Wetering, Linda M Berg, Roberto Verado, Dipti Vijayan, Ilya E Vorontsov, Wyeth W Wasserman, Shoko Watanabe, Christine A Wells, Louise N Winteringham, Ernst Wolvetang, Emily J Wood, Yoko Yamaguchi, Masayuki Yamamoto, Misako Yoneda, Yohei Yonekura, Shigehiro Yoshida, Susan E Zabierowski, Peter G Zhang, Xiaobei Zhao, Silvia Zucchelli, Kim M Summers, Harukazu Suzuki, Carsten O Daub, Jun Kawai, Peter Heutink, Winston Hide, Tom C Freeman, Boris Lenhard, Vladimir B Bajic, Martin S Taylor, Vsevolod J Makeev, Albin Sandelin, David A Hume, Piero Carninci, Yoshihide Hayashizaki' : '259 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">259 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Mar 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/nature13182" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Regulated transcription controls the diversity, developmental pathways and spatial organization of the hundreds of cell types that make up a mammal. Using single-molecule cDNA sequencing, we mapped transcription start sites (TSSs) and their usage in human and mouse primary cells, cell lines and tissues to produce a comprehensive overview of mammalian gene expression across the human body. We find that few genes are truly ’housekeeping’, whereas many mammalian promoters are composite entities composed of several closely separated TSSs, with independent cell-type-specific expression profiles. TSSs specific to different cell types evolve at different rates, whereas promoters of broadly expressed genes are the most conserved. Promoter-based expression analysis reveals key transcription factors defining cell states and links them to binding-site motifs. The functions of identified novel transcripts can be predicted by coexpression and sample ontology enrichment analyses. The functional annotation of the mammalian genome 5 (FANTOM5) project provides comprehensive expression profiles and functional annotation of mammalian cell-type-specific transcriptomes with wide applications in biomedical research.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Andersson2014-bh" class="col-sm-10"> <div class="title">An atlas of active enhancers across human cell types and tissues</div> <div class="author"> Robin Andersson, Claudia Gebhard, Irene Miguel-Escalada, and <span class="more-authors" title="click to view 48 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '48 more authors' ? 'Ilka Hoof, Jette Bornholdt, Mette Boyd, Yun Chen, Xiaobei Zhao, Christian Schmidl, Takahiro Suzuki, Evgenia Ntini, Erik Arner, Eivind Valen, Kang Li, Lucia Schwarzfischer, Dagmar Glatz, Johanna Raithel, Berit Lilje, Nicolas Rapin, Frederik Otzen Bagger, Mette Jørgensen, Peter Refsing Andersen, Nicolas Bertin, Owen Rackham, A Maxwell Burroughs, J Kenneth Baillie, Yuri Ishizu, Yuri Shimizu, Erina Furuhata, Shiori Maeda, Yutaka Negishi, Christopher J Mungall, Terrence F Meehan, Timo Lassmann, Masayoshi Itoh, Hideya Kawaji, Naoto Kondo, Jun Kawai, Andreas Lennartsson, Carsten O Daub, Peter Heutink, David A Hume, Torben Heick Jensen, Harukazu Suzuki, Yoshihide Hayashizaki, Ferenc Müller, Consortium, The FANTOM, Alistair R R Forrest, Piero Carninci, Michael Rehli, Albin Sandelin' : '48 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">48 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Mar 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/nature12787" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Vitezic2014-av" class="col-sm-10"> <div class="title">CAGE-defined promoter regions of the genes implicated in Rett Syndrome</div> <div class="author"> Morana Vitezic, Nicolas Bertin, Robin Andersson, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Leonard Lipovich, Hideya Kawaji, Timo Lassmann, Albin Sandelin, Peter Heutink, Dan Goldowitz, Thomas Ha, Peter Zhang, Annarita Patrizi, Michela Fagiolini, Alistair Rr Forrest, Piero Carninci, Alka Saxena, The FANTOM Consortium' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>BMC Genomics</em>, Jan 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/1471-2164-15-1177" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"><li> <div class="row"> <div id="Ntini2013-jx" class="col-sm-10"> <div class="title">Polyadenylation site–induced decay of upstream transcripts enforces promoter directionality</div> <div class="author"> Evgenia Ntini, Aino I Järvelin, Jette Bornholdt, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Yun Chen, Mette Boyd, Mette Jørgensen, Robin Andersson, Ilka Hoof, Aleks Schein, Peter R Andersen, Pia K Andersen, Pascal Preker, Eivind Valen, Xiaobei Zhao, Vicent Pelechano, Lars M Steinmetz, Albin Sandelin, Torben Heick Jensen' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Nat. Struct. Mol. Biol.</em>, Jul 2013 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/nsmb.2640" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"><li> <div class="row"> <div id="Enroth2012-ww" class="col-sm-10"> <div class="title">A strand specific high resolution normalization method for chip-sequencing data employing multiple experimental control measurements</div> <div class="author"> Stefan Enroth, Claes R Andersson, Robin Andersson, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Claes Wadelius, Mats G Gustafsson, Jan Komorowski' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Algorithms Mol. Biol.</em>, Jan 2012 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/1748-7188-7-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"><li> <div class="row"> <div id="Enroth2011-dn" class="col-sm-10"> <div class="title">Cancer associated epigenetic transitions identified by genome-wide histone methylation binding profiles in human colorectal cancer samples and paired normal mucosa</div> <div class="author"> Stefan Enroth, Alvaro Rada-Iglesisas, Robin Andersson, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Ola Wallerman, Alkwin Wanders, Lars Påhlman, Jan Komorowski, Claes Wadelius' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>BMC Cancer</em>, Jan 2011 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/1471-2407-11-450" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Poplawski2010-ia" class="col-sm-10"> <div class="title">Frequent genetic differences between matched primary and metastatic breast cancer provide an approach to identification of biomarkers for disease progression</div> <div class="author"> Andrzej B Popławski, Michał Jankowski, Stephen W Erickson, and <span class="more-authors" title="click to view 18 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '18 more authors' ? 'Teresita Ståhl, E Christopher Partridge, Chiquito Crasto, Jingyu Guo, John Gibson, Uwe Menzel, Carl Eg Bruder, Aneta Kaczmarczyk, Magdalena Benetkiewicz, Robin Andersson, Johanna Sandgren, Barbara Zegarska, Dariusz Bała, Ewa Srutek, David B Allison, Arkadiusz Piotrowski, Wojciech Zegarski, Jan P Dumanski' : '18 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">18 more authors</span> </div> <div class="periodical"> <em>Eur. J. Hum. Genet.</em>, "6 " # "jan" 2010 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/ejhg.2009.230" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Enroth2010-ao" class="col-sm-10"> <div class="title">SICTIN: Rapid footprinting of massively parallel sequencing data</div> <div class="author"> Stefan Enroth, Robin Andersson, Claes Wadelius, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Jan Komorowski' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>BioData Min.</em>, Jan 2010 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1186/1756-0381-3-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Sandgren2010-xn" class="col-sm-10"> <div class="title">Integrative epigenomic and genomic analysis of malignant pheochromocytoma</div> <div class="author"> Johanna Sandgren, Robin Andersson, Alvaro Rada-Iglesias, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Stefan Enroth, Göran Akerström, Jan P Dumanski, Jan Komorowski, Gunnar Westin, Claes Wadelius' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Exp. Mol. Med.</em>, Jan 2010 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.3858/emm.2010.42.7.050" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2009</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Mantripragada2009-wz" class="col-sm-10"> <div class="title">Genome-wide high-resolution analysis of DNA copy number alterations in NF1-associated malignant peripheral nerve sheath tumors using 32K BAC array</div> <div class="author"> Kiran K Mantripragada, Teresita Ståhl, Chris Patridge, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Uwe Menzel, Robin Andersson, Nadia Chuzhanova, Lan Kluwe, Abhijit Guha, Victor Mautner, Jan P Dumanski, Meena Upadhyaya' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Genes Chromosomes Cancer</em>, Oct 2009 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/gcc.20695" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Neurofibromatosis Type I (NF1) is an autosomal dominant disorder characterized by the development of both benign and malignant tumors. The lifetime risk for developing a malignant peripheral nerve sheath tumor (MPNST) in NF1 patients is approximately 10% with poor survival rates. To date, the molecular basis of MPNST development remains unclear. Here, we report the first genome-wide and high-resolution analysis of DNA copy number alterations in MPNST using the 32K bacterial artificial chromosome microarray on a series of 24 MPNSTs and three neurofibroma samples. In the benign neurofibromas, apart from loss of one copy of the NF1 gene and copy number polymorphisms, no other changes were found. The profiles of malignant samples, however, revealed specific loss of chromosomal regions including 1p35-33, 1p21, 9p21.3, 10q25, 11q22-23, 17q11, and 20p12.2 as well as gain of 1q25, 3p26, 3q13, 5p12, 5q11.2-q14, 5q21-23, 5q31-33, 6p23-p21, 6p12, 6q15, 6q23-q24, 7p22, 7p14-p13, 7q21, 7q36, 8q22-q24, 14q22, and 17q21-q25. Copy number gains were more frequent than deletions in the MPNST samples (62% vs. 38%). The genes resident within common regions of gain were NEDL1 (7p14), AP3B1 (5q14.1), and CUL1 (7q36.1) and these were identified in &gt;63% MPNSTs. The most frequently deleted locus encompassed CDKN2A, CDKN2B, and MTAP genes on 9p21.3 (33% cases). These genes have previously been implicated in other cancer conditions and therefore, should be considered for their therapeutic, prognostic, and diagnostic relevance in NF1 tumorigenesis.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Bjorkholm2009-yp" class="col-sm-10"> <div class="title">Using multi-data hidden Markov models trained on local neighborhoods of protein structure to predict residue-residue contacts</div> <div class="author"> Patrik Björkholm, Pawel Daniluk, Andriy Kryshtafovych, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Krzysztof Fidelis, Robin Andersson, Torgeir R Hvidsten' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Bioinformatics</em>, May 2009 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/bioinformatics/btp149" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>MOTIVATION: Correct prediction of residue-residue contacts in proteins that lack good templates with known structure would take ab initio protein structure prediction a large step forward. The lack of correct contacts, and in particular long-range contacts, is considered the main reason why these methods often fail. RESULTS: We propose a novel hidden Markov model (HMM)-based method for predicting residue-residue contacts from protein sequences using as training data homologous sequences, predicted secondary structure and a library of local neighborhoods (local descriptors of protein structure). The library consists of recurring structural entities incorporating short-, medium- and long-range interactions and is general enough to reassemble the cores of nearly all proteins in the PDB. The method is tested on an external test set of 606 domains with no significant sequence similarity to the training set as well as 151 domains with SCOP folds not present in the training set. Considering the top 0.2 x L predictions (L = sequence length), our HMMs obtained an accuracy of 22.8% for long-range interactions in new fold targets, and an average accuracy of 28.6% for long-, medium- and short-range contacts. This is a significant performance increase over currently available methods when comparing against results published in the literature. AVAILABILITY: http://predictioncenter.org/Services/FragHMMent/.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Rada-Iglesias2009-zj" class="col-sm-10"> <div class="title">Histone H3 lysine 27 trimethylation in adult differentiated colon associated to cancer DNA hypermethylation</div> <div class="author"> Alvaro Rada-Iglesias, Stefan Enroth, Robin Andersson, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Alkwin Wanders, Lars Påhlman, Jan Komorowski, Claes Wadelius' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Epigenetics</em>, Feb 2009 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.4161/epi.4.2.8038" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>DNA hypermethylation of gene promoters is a common epigenetic alteration occurring in cancer cells. However, little is known about the mechanisms instructing these cancer-specific DNA hypermethylation events. Recent reports have suggested that genes bound by polycomb/Histone H3 lysine 27 trimethylation (H3K27me3) in embryonic stem (ES) cells are frequent targets for cancer-specific DNA hypermethylation. This polycomb-premarking is assumed to be restrained to ES cells, even though almost no polycomb/H3K27me3 binding profiles are available for differentiated tissues. We generated H3K27me3 profiles in human normal colon and they significantly overlapped with those of ES cells and genes hypermethylated in colorectal cancer (CRC). Moreover, colon H3K27me3 was more restricted to genes hypermethylated in CRC, while ES H3K27me3 was also common in genes hypermethylated in other tumors. Therefore, the suggested polycomb pre-marking of genes for cancer DNA hypermethylation is not necessarily limited to ES or early precursor cells but can occur later in differentiated tissues.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Piotrowski2008-jg" class="col-sm-10"> <div class="title">Somatic mosaicism for copy number variation in differentiated human tissues</div> <div class="author"> Arkadiusz Piotrowski, Carl E G Bruder, Robin Andersson, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Teresita Diaz Ståhl, Uwe Menzel, Johanna Sandgren, Andrzej Poplawski, Desiree Tell, Chiquito Crasto, Adam Bogdan, Rafal Bartoszewski, Zsuzsa Bebok, Maciej Krzyzanowski, Zbigniew Jankowski, E Christopher Partridge, Jan Komorowski, Jan P Dumanski' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Hum. Mutat.</em>, Sep 2008 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1002/humu.20815" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div id="Diaz-de-Stahl2008-cb" class="col-sm-10"> <div class="title">Profiling of copy number variations (CNVs) in healthy individuals from three ethnic groups using a human genome 32 K BAC-clone-based array</div> <div class="author"> Teresita Ståhl, Johanna Sandgren, Arkadiusz Piotrowski, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Helena Nord, Robin Andersson, Uwe Menzel, Adam Bogdan, Ann-Charlotte Thuresson, Andrzej Poplawski, Desiree Tell, Caisa M Hansson, Amir I Elshafie, Gehad Elghazali, Stephan Imreh, Magnus Nordenskjöld, Meena Upadhyaya, Jan Komorowski, Carl E G Bruder, Jan P Dumanski' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>Hum. Mutat.</em>, Mar 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/humu.20659" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>To further explore the extent of structural large-scale variation in the human genome, we assessed copy number variations (CNVs) in a series of 71 healthy subjects from three ethnic groups. CNVs were analyzed using comparative genomic hybridization (CGH) to a BAC array covering the human genome, using DNA extracted from peripheral blood, thus avoiding any culture-induced rearrangements. By applying a newly developed computational algorithm based on Hidden Markov modeling, we identified 1,078 autosomal CNVs, including at least two neighboring/overlapping BACs, which represent 315 distinct regions. The average size of the sequence polymorphisms was approximately 350 kb and involved in total approximately 117 Mb or approximately 3.5% of the genome. Gains were about four times more common than deletions, and segmental duplications (SDs) were overrepresented, especially in larger deletion variants. This strengthens the notion that SDs often define hotspots of chromosomal rearrangements. Over 60% of the identified autosomal rearrangements match previously reported CNVs, recognized with various platforms. However, results from chromosome X do not agree well with the previously annotated CNVs. Furthermore, data from single BACs deviating in copy number suggest that our above estimate of total variation is conservative. This report contributes to the establishment of the common baseline for CNV, which is an important resource in human genetics.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Bruder2008-aq" class="col-sm-10"> <div class="title">Phenotypically Concordant and Discordant Monozygotic Twins Display Different DNA Copy-Number-Variation Profiles</div> <div class="author"> Carl E G Bruder, Arkadiusz Piotrowski, Antoinet A C J Gijsbers, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Robin Andersson, Stephen Erickson, Teresita Ståhl, Uwe Menzel, Johanna Sandgren, Desiree Tell, Andrzej Poplawski, Michael Crowley, Chiquito Crasto, E Christopher Partridge, Hemant Tiwari, David B Allison, Jan Komorowski, Gert-Jan B Ommen, Dorret I Boomsma, Nancy L Pedersen, Johan T Dunnen, Karin Wirdefeldt, Jan P Dumanski' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Am. J. Hum. Genet.</em>, Mar 2008 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1016/j.ajhg.2007.12.011" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2007</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Mikhail2007-vq" class="col-sm-10"> <div class="title">A previously unrecognized microdeletion syndrome on chromosome 22 band q11.2 encompassing the BCR gene</div> <div class="author"> Fady M Mikhail, Maria Descartes, Arkadiusz Piotrowski, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Robin Andersson, Teresita Ståhl, Jan Komorowski, Carl E G Bruder, Jan P Dumanski, Andrew J Carroll' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Am. J. Med. Genet. A</em>, Sep 2007 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/ajmg.a.31882" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Susceptibility of the chromosome 22q11.2 region to rearrangements has been recognized on the basis of common clinical disorders such as the DiGeorge/velocardiofacial syndrome (DG/VCFs). Recent evidence has implicated low-copy repeats (LCRs); also known as segmental duplications; on 22q as mediators of nonallelic homologous recombination (NAHR) that result in rearrangements of 22q11.2. It has been shown that both deletion and duplication events can occur as a result of NAHR caused by unequal crossover of LCRs. Here we report on the clinical, cytogenetic and array CGH studies of a 15-year-old Hispanic boy with history of learning and behavior problems. We suggest that he represents a previously unrecognized microdeletion syndrome on chromosome 22 band q11.2 just telomeric to the DG/VCFs typically deleted region and encompassing the BCR gene. Using a 32K BAC array CGH chip we were able to refine and precisely narrow the breakpoints of this microdeletion, which was estimated to be 1.55-1.92 Mb in size and to span approximately 20 genes. This microdeletion region is flanked by LCR clusters containing several modules with a very high degree of sequence homology (&gt;95%), and therefore could play a causal role in its origin.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Mikhail2007-ai" class="col-sm-10"> <div class="title">Overlapping phenotype of Wolf-Hirschhorn and Beckwith-Wiedemann syndromes in a girl with der(4)t(4;11)(pter;pter)</div> <div class="author"> Fady M Mikhail, Achara Sathienkijkanchai, Nathaniel H Robin, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Sandra Prucka, Julie Sanford Biggerstaff, Jan Komorowski, Robin Andersson, Carl E G Bruder, Arkadiusz Piotrowski, Teresita Ståhl, Jan P Dumanski, Andrew J Carroll' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Am. J. Med. Genet. A</em>, Aug 2007 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/ajmg.a.31821" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We report on an 8-month-old girl with a novel unbalanced chromosomal rearrangement, consisting of a terminal deletion of 4p and a paternal duplication of terminal 11p. Each of these is associated with the well-known clinical phenotypes of Wolf-Hirschhorn syndrome (WHS) and Beckwith-Wiedemann syndrome (BWS), respectively. She presented for clinical evaluation of dysmorphic facial features, developmental delay, atrial septal defect (ASD), and left hydronephrosis. High-resolution cytogenetic analysis revealed a normal female karyotype, but subtelomeric fluorescence in situ hybridization (FISH) analysis revealed a der(4)t(4;11)(pter;pter). Both FISH and microarray CGH studies clearly demonstrated that the WHS critical regions 1 and 2 were deleted, and that the BWS imprinted domains (ID) 1 and 2 were duplicated on the der(4). Parental chromosome analysis revealed that the father carried a cryptic balanced t(4;11)(pter;pter). As expected, our patient manifests findings of both WHS (a growth retardation syndrome) and BWS (an overgrowth syndrome). We compare her unique phenotypic features with those that have been reported for both syndromes.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Andersson Lab. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-XQNNEFWBCH"></script> <script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'G-XQNNEFWBCH');
  </script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> </body> </html>